<!DOCTYPE html>
<html lang="en">

  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>phylogeography</title>

    <!-- Bootstrap core CSS -->
    <link href="../vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="../css/1-col-portfolio.css" rel="stylesheet">

  </head>

  <body>

    <!-- Navigation -->
    <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
      <div class="container">
        <a class="navbar-brand" href="../index.html">Justin C. Bagley - Ecological & Evolutionary Genetics</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
          <ul class="navbar-nav ml-auto">
            <li class="nav-item">
              <a class="nav-link" href="../index.html">home</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="about.html">about</a>
            </li>
            <li class="nav-item active">
              <a class="nav-link" href="research.html">research
                <span class="sr-only">(current)</span>
              </a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="pubs.html">publications</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="talks.html">talks</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="software.html">software</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="teaching.html">teaching</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="tutorials.html">tutorials</a>
            </li>
          </ul>
        </div>
      </div>
    </nav>


<!-- Page Content -->
    <div class="container">

	<div></div>
	<p></p>

<!-- Phylogeography Content -->
    <div class="container-narrow">
    <a name="top"></a>
      <!-- Page Heading -->
      <!-- <h1 class="my-4">Recent Projects -->
        <!-- OPTION BELOW: small text next to h1 heading, e.g. subheading) -->
        <!-- <small>(on-going and published)</small> -->
      <h1 class="my-4">phylogeography
      </h1>

<h2 id="what-is-phylogeography">what is phylogeography?</h2>

	<blockquote cite="">
	<p><em>[G]eology and biogeography are both parts of natural history and, if they represent the independent and dependent variables respectively in a cause and effect relationship, … they can be reciprocally illuminating.</em></p>
	<cite>–Rosen (1978, p. 776)</cite><br />
	</blockquote>
		<!-- <p>&nbsp;</p> -->
	<blockquote cite="">
	<p><em>[C]omparative phylogeographic assessments within each of multiple codistributed species … offer perhaps the greatest hope for significant advances in understanding how … the demographic and natural histories of populations, can influence intraspecific phylogeographic patterns.</em></p>
	<cite>–Avise (1998, pp. 376-377)</cite>
	</blockquote>
		<p>&nbsp;</p>

<p>In the past, molecular biology informed the methods of population genetics, while biogeographers operated independently of these fields.  However, since the 20th Century molecular genetics revolution in systematics and population genetics, it is the tools of these fields that dictate methods for analyzing genetic data from natural populations, and this in turn has sparked a new field of ‘molecular biogeography.’  The growth of the ‘new’ biogeography is widely acknowledged to have particularly benefited from the rapid development of <a href="http://en.wikipedia.org/wiki/Phylogeography">phylogeography</a>. Yet, what is phylogeography?</p>
<p>Phylogeography is a relatively young and integrative field of science that uses molecular data to infer the processes influencing geographical distributions of genetic lineages within and among species, especially at the intraspecific level (Avise et al. 1987; Avise 2000). The phylogeography literature base has grown remarkably fast over the nearly three decades since the birth of the field, yielding many insights into the histories of biotas in different ecosystems across the globe, including cryptic and often temporally ‘deep’ genealogical divergences within many species ranges (e.g. reviewed by Beheregaray 2008; Knowles 2009). By linking these patterns of population divergence with data on geographical barriers, earth history events, distribution models, and speciation processes, phylogeography permits identifying and testing whether and over which temporal scales historical and recurrent processes have shaped intraspecific diversification in an area. Indeed, the reciprocal illumination that Rosen (1978; quoted above) envisioned for vicariance biogeography can also be achieved through phylogeography, as phylogeography provides means of inferring how species responded to geological processes without reliance on phylogenetic structure or areas of endemism required by other historical biogeography methods (Zink 2002); for example, historical processes from the unobservable past can still be inferred even when genetic breaks revealed by selectively neutral markers do not correspond to any present-day geographical barrier.</p>
<p align="center"><img src="http://www.justinbagley.org/wp-content/uploads/2015/11/phylogeography_bloomqSum.png" title="Bloomquist et al. 2010" alt="Bloomquist et al. 2010" width="280" class="center"></img></p>
<p>Moreover, because population or species demographic histories influence the shapes of their gene genealogies, we can use coalescent theory (a stochastic, backwards-in-time theory of genealogical processes; Kingman 1982) to estimate their underlying genealogies (e.g. by simulating a distribution of them) as well as population parameters describing their histories (e.g. past changes in population sizes) while accounting for various population genetic processes (e.g. subdivision, speciation, mutation, recombination; reviewed by Kuhner 2009). An important paradigm shift in phylogeography has been the realization that phylogeographic inferences are improved by developing and discriminating among competing coalescent-based demographic models using statistical methods that account for the stochasticity of genetic processes (Knowles &amp; Maddison 2002; Knowles 2009; Nielsen &amp; Beaumont 2009).</p>
<p>The above graphic (<em>above-center</em>) summarizes several 'paths' to phylogeographic inference reviewed by Bloomquist et al. (2010), including comparative, spatial diffusion, and statistical population genetics approaches, which each view the recovery of species demographic histories from different perspectives involving different forms of inferences (e.g. pattern-based methods; diffusion-model based inferences; and full-likelihood, full-Bayesian, and ABC model-based analyses).</p>

<h2 id="my-prior-research-in-phylogeography">my prior research in Phylogeography</strong></h2>
<p>Given I have discussed my ongoing research in phylogeography elsewhere in my About, Research, and Collaborators pages, this section provides a summary of my prior work in this fascinating and growing field, including studies conducted along two main lines of research:</p>

	<ul>
	<li><strong>congeneric and comparative phylogeography</strong></li>
	<li><strong>model-based inference of species responses to Pleistocene climate and sea level change</strong></li>
	</ul>

<h5 id="congeneric-and-comparative-phylogeography"><em>congeneric and comparative phylogeography</em></h5>

<p align="center"><a href="http://www.justinbagley.org/wp-content/uploads/2013/11/Spotted-bass-M-punctulatus.png"><img src="http://www.justinbagley.org/wp-content/uploads/2013/11/Spotted-bass-M-punctulatus.png" alt="Spotted bass M punctulatus" /></a></p><br />
<p>While working towards my Master’s in Biology at UA, I asked what unique insights congeneric phylogeographical sampling (within a genus) might yield into understanding the phylogenetics and species limits of North American black basses (<a href="http://www.fishbase.org/summary/FamilySummary.php?ID=302">Centrarchidae</a>:<em> Micropterus</em>), an iconic group of game fishes with a complex systematic and taxonomic history_.<em>  Based on sampling from across the geographical ranges of black bass species, we found multiple cases of mitochondrial introgression/hybridization events and a novel Gulf Coast clade representing a new lineage to science; thus phylogeography reveals a more complex view of black bass evolution highlighting population-level processes (<a href="http://www.justinbagley.org/publications">Bagley et al. 2011, Biol. J. Linn. Soc.</a>).  I also combined mtDNA phylogeography, microsatellite DNA population genetics, and morphometric analysis to study Alabama Bass (</em>M. henshalli_).  I found that, despite evidence for hybridization with Redeye Bass (<em>M. coosae</em>), <em>M. henshalli</em> is overall genetically and morphologically distinct, particularly within the Tallapoosa River system (manuscript in prep.).</p>
<p><a href="http://en.wikipedia.org/wiki/Phylogeography#Comparative_phylogeography">Comparative phylogeography</a> provides a basis for more robust historical inferences (e.g. of historical biogeographical scenarios for a region) by searching for replicated geographical patterns across multiple codistributed species (e.g. Bermingham and Martin 1998), yielding insights into community assembly and diversification not possible through single-species phylogeography alone.</p>
<p align="center"><a href="http://www.justinbagley.org/wp-content/uploads/2013/11/PIA03364_modest-w-Pmex-Anigro-Acult.png"><img src="http://www.justinbagley.org/wp-content/uploads/2013/11/PIA03364_modest-w-Pmex-Anigro-Acult.png" alt="PIA03364_modest w Pmex Anigro Acult" width="280"/></a></p>
<p>My Ph.D. dissertation work at BYU focused on using an integrative approach coupling comparative phylogeography with species delimitation to improve our understanding of the biogeography and species limits in complex tropical landscapes.   The Central American (CA) land bridge and its freshwater stream fish assemblage provide an ideal study system for this research combining a complex/recent physical history with rich geomorphology and high endemism, including several codistributed lineages of freshwater fishes.  My dissertation (1) developed and tested biogeographical hypotheses for CA through a synthesis of geological and phylogeographical data (<a href="http://www.justinbagley.org/publications">Bagley &amp; Johnson 2014a, Biol. Rev.</a>); (2) analyzed single-species/lineage phylogeography datasets for up to eight lineages, including <a href="http://www.fishbase.org/summary/FamilySummary.php?ID=216">Poeciliidae</a> (five lineages), <a href="http://www.fishbase.org/summary/FamilySummary.php?ID=102">Characidae</a> (two species), and <a href="http://www.fishbase.org/summary/FamilySummary.php?ID=349">Cichlidae</a> (one lineage, 4 spp.); and (3) ultimately scaled my own data and data curated from online repositories up to comparative analyses of 8 fish species/lineages (<a href="http://www.justinbagley.org/publications">Bagley &amp; Johnson 2014b, Ecol. Evol.</a>; other manuscripts in prep.).  Given a moderate to high degree of taxonomic uncertainty for various groups of Neotropical fishes I studied, this work also evolved to entail a component of species delimitation.  Specifically, I used mollies in the <em>Poecilia sphenops</em> species complex_ _(Poeciliidae) as a model system for delimiting species and studying processes of diversification in widespread, morphologically conserved groups of Neotropical fishes, through coalescent-based analyses of multilocus genetic datasets.  Therefore, the over-arching goals of my Ph.D. were to evaluate the impacts of historical processes (e.g. geological, climatic events) on geographical distributions of genetic lineages, to understand the relationship between species ecological differences and their phylogeographic histories, and to test and refine Central American freshwater fish species boundaries.  I also worked with my Ph.D. advisor, <a href="http://www.justinbagley.org/collaborators">Jerry Johnson</a>, to build phenotypic datasets that will allow me to compare morphometric and life history variation with phylogeographic divergence using spatial-pattern analyses, to test hypotheses about selection and species limits. </p>
<p>While at BYU, I also developed research on the biogeography and evolution of Australian freshwater fishes in collaboration with <a href="http://www.justinbagley.org/collaborators">Peter Unmack</a>.  By melding species tree analyses with phylogeographic inferences from mitochondrial and nuclear loci, we documented an Oligocene-Miocene origin for most species of galaxiid fishes in the genus <em>Galaxiella </em>(<a href="http://www.fishbase.org/summary/FamilySummary.php?ID=79">Galaxiidae</a>)<em>,</em> high genetic diversity, and strong evidence for a cryptic species within the threatened dwarf galaxias, <em>G. pusilla</em> (<a href="http://www.justinbagley.org/publications">Unmack et al. 2012, PLoS ONE</a>).  Our findings weigh importantly on conserving this threatened clade of fishes, e.g. long-term planning should preserve multiple populations with unique alleles, in addition to the multiple ancient species in the group.  I am currently working to expand my work in Australia to help develop a comparative perspective on the phylogeographic diversification of Australian freshwater fish assemblages.  With Dr. Unmack, I am also presently working to expand phylogenetic systematics work on iconic game fishes, through a multilocus phylogenetic analysis of temperate perches in the family <a href="http://fishbase.org/Summary/FamilySummary.php?ID=288">Percichthyidae</a>, which are the largest fish species historically present in most temperate Australian freshwater habitats.</p>

<h5 id="model-based-inference-of-species-responses-to-pleistocene-climate-and-sea-level-change"><em>model-based inference of species responses to Pleistocene climate and sea level change</em></h5>
<p>A product of the past two decades of analytical revolutions in biology has been the development of novel model-based statistical tools for (1) inferring parameters of ecological niches and predicting species past, present and future distributions, and (2) using coalescent simulations to arrive at objective rather than ad hoc inferences of processes influencing species phylogeographic histories.  I am interested in using these toolsets, i.e. ecological niche modeling and statistical phylogeography, to test hypotheses and infer population history (e.g. lineage spread and range shifting), particularly the responses of species from unglaciated subtropical and tropical areas to climatic fluctuations of the late Pleistocene glaciations.  For example, in our work on <em>Galaxiella</em> fishes, we documented a shared demographic-growth response across taxa associated with relaxation of aridity during the late Pleistocene, based on statistical comparisons of Bayesian demographic models using Bayes factors (<a href="http://www.justinbagley.org/publications">Unmack et al. 2012, PLoS ONE</a>).  I then revisited the southeastern U.S. freshwater fish fauna to test the potential impact of expansion-contraction cycles on species ranges, hence patterns of genetic structure, during the last glacial cycle.  Paleoclimatic modeling, phylogeography, and coalescent simulations using sequence data from least killifish, <em>Heterandria formosa</em>, suggested that this cold-intolerant species contracted its range to a peninsular Florida refugium during the Last Glacial Maximum (<a href="http://www.justinbagley.org/publications">Bagley et al. 2013, BMC Evol. Biol.</a>).  We also found evidence for nuclear-mitochondrial discordances indicating evidence for historical gene flow and sex-biased migration.</p>

<h5 id="section"> </h5>
<h2 id="future-phylogeography-research-directions">future phylogeography research directions</h2>
<p>While the review of my previous work on phylogeography above is informative, it is exciting to speculate about potential future research directions in this field. One thing I am very excited about regarding the future of my phylogeographical research is that I intend to continue the past lines of research above, while maintaining a ‘big picture’ view, by focusing on community and ecosystem-level syntheses.  In addition, I look forward to developing the phylogeography arm of my research program along several lines of research, including:</p>

	<blockquote>
	<ol style="list-style-type: decimal">
	<li>using model-based comparative phylogeography to infer community assembly and diversification,
	<li>using landscape genetics and ecological niche modeling to explore the relative roles of ecology and geography in shaping genetic isolation in natural populations (with empirical analyses of Central American terrestrial and freshwater species), and 
	<li>using conservation phylogeography to aid resolving the 'Linnean' and 'Wallacean' shortfalls in biogeography and conservation, and to inform conservation management and policy, particularly in the Central American and South American Neotropics.
	</blockquote>
	</li>
	</ol>

<p>While I intend to conduct this research in my main geographical areas of focus, the South American (Brazil) and Central American Neotropics, I would be interested in pursuing conservation biogeography in my other study systems discussed on my Research page, or any other region of the world with a highly endemic and threatened biota.  However, by extending phylogeography to inform management I will help us not only understand, but also conserve, the patterns and processes of diversification in unique Neotropical (and other) regions of the world.</p>

<p><font size="-1"><a href="#top">top</a></font></p>

<hr>

<p><strong>References</strong></p>
<p>Avise JC (1998) The history and purview of phylogeography: a personal reflection. Molecular Ecology, 7, 371-379.</p>
<p>Avise JC (2000) Phylogeography: the History and Formation of Species. Harvard University Press, Cambridge, Massachusetts.</p>
<p>Avise JC, Arnold J, Ball RM, Bermingham E, Lamb T, Neigel JE, Reeb CA, Saunders NC (1987) Intraspecific phylogeography: the mitochondrial DNA bridge between population genetics and systematics. Annual Review of Ecology and Systematics, 18, 489-522.</p>
<p>Bagley JC, Johnson JB (2014a) Phylogeography of the lower Central American Neotropics: diversification between two continents and between two seas. Biological Reviews, 89(4), 767-790.</p>
<p>Bagley JC, Johnson JB (2014b) Testing for shared biogeographic history in the lower Central American freshwater fish assemblage using comparative phylogeography: concerted, independent, or multiple evolutionary responses? Ecology and Evolution, 4(9), 1686-1705.</p>
<p>Bagley JC, Mayden RL, Roe KJ, Holznagel W, Harris PM (2011). Congeneric phylogeographical sampling reveals polyphyly and novel biodiversity within black basses (Centrarchidae: <em>Micropterus</em>). Biological Journal of the Linnean Society, 104(2), 346-363.</p>
<p>Bagley JC, Sandel M, Travis J, Lozano-Vilano M de L, Johnson JB (2013) Paleoclimatic modeling and phylogeography of least killifish, <em>Heterandria formosa</em>: insights into Pleistocene expansion-contraction dynamics and evolutionary history of North American Coastal Plain freshwater biota. BMC Evolutionary Biology, 13, 223.</p>
<p>Unmack PJ, Bagley JC, Adams MD, Hammer MD, Johnson JB (2012) Molecular phylogeny and phylogeography of the Australian freshwater fish genus <em>Galaxiella</em> (Teleostei: Galaxiidae), with an emphasis on dwarf galaxias (<em>G. pusilla</em>). PLoS One, 7(6), e38433.</p>
<p>Beheregaray LB (2008) Twenty years of phylogeography: the state of the field and the challenges for the Southern Hemisphere. Molecular Ecology, 17, 3754-3774.</p>
<p>Bermingham E, Martin AP (1998) Comparative mtDNA phylogeography of neotropical freshwater fishes: testing shared history to infer the evolutionary landscape of lower Central America. Molecular Ecology, 7, 499-517.</p>
<p>Bloomquist EW, Lemey P, Suchard MA (2010) Three roads diverged? Routes to phylogeographic inference. Trends in Ecology &amp; Evolution, 25, 626-632.</p>
<p>Knowles LL (2009) Statistical phylogeography. Annual Review in Ecology, Evolution, and Systematics, 40, 593-612.</p>
<p>Rosen DE (1978) Vicariant patterns and historical explanation in biogeography. Systematic Zoology, 27, 159-188.</p>
<p>Zink RM (2002) Methods in comparative phylogeography, and their application to studying evolution in the North American aridlands. Integrative and Comparative Biology, 42, 953-959.
</p>

<p><font size="-1"><a href="#top">top</a></font></p>

<hr>
    </div>
    <!-- /.container-narrow -->
			
	<div></div>

	<!-- <hr> -->
    </div>
    <!-- /.container (Page Content) -->


    <!-- Footer -->
    <footer class="py-5 bg-dark">
      <div class="container">
        <p class="m-0 text-center text-white">Copyright &copy; 2018 Justin C. Bagley</p>
      </div>
      <!-- /.container -->
    </footer>

    <!-- Bootstrap core JavaScript -->
    <script src="../vendor/jquery/jquery.min.js"></script>
    <script src="../vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

  </body>

</html>

