<!DOCTYPE html>
<html lang="en">

  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>protocols</title>

    <!-- Bootstrap core CSS -->
    <link href="../vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="../css/1-col-portfolio.css" rel="stylesheet">

  </head>

  <body>

    <!-- Navigation -->
    <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
      <div class="container">
        <a class="navbar-brand" href="../index.html">Justin C. Bagley - Ecological & Evolutionary Genetics</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
          <ul class="navbar-nav ml-auto">
            <li class="nav-item">
              <a class="nav-link" href="../index.html">home</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="about.html">about</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="research.html">research</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="pubs.html">publications</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="talks.html">talks</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="software.html">software</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="teaching.html">teaching</a>
            </li>
            <li class="nav-item active">
              <a class="nav-link" href="protocols.html">protocols
                <span class="sr-only">(current)</span>
	      </a>
            </li>
          </ul>
        </div>
      </div>
    </nav>


<!-- Page Content -->
    <div class="container-narrow">

      <!-- Page Heading -->
      <h1 class="my-4">protocols</h1>
        


<!-- <h3 id="molecular-laboratory-protocols">Molecular Laboratory Protocols</h3>  -->

<h5 id="protocols-for-getting-and-analyzing-dna-data-for-phylogenetics-and-phylogeography">
	<em>Protocols for obtaining and analyzing DNA/NGS data for phylogenetics and phylogeography</em>
</h5>

<nav class="navbar navbar-expand-lg navbar-light bg-light">
  <a class="navbar-brand" href="#">Navbar</a>
  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button>
  <div class="collapse navbar-collapse" id="navbarNavDropdown">
    <ul class="navbar-nav">
      <li class="nav-item active">
        <a class="nav-link" href="#molecular-laboratory-supplies">Supplies<span class="sr-only">(current)</span></a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#DNA-extraction">DNA extraction</a>
      </li>
      <li class="nav-item dropdown">
<!--
	<a class="nav-link dropdown-toggle" href="#PCR_and_Sanger" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">PCR &amp; Sanger sequencing</a> 
-->
	<a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">PCR &amp; Sanger sequencing</a>
        <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
	  <a class="dropdown-item" href="#millipore">Millipore PCR cleanup</a>
	  <a class="dropdown-item" href="#exosapit">ExoSAP-IT PCR cleanup</a>
	  <a class="dropdown-item" href="#bigdye">Big Dye cycle sequencing</a>
	  <a class="dropdown-item" href="#sephadex">Sephadex sequencing cleanup</a>
	  <a class="dropdown-item" href="#etohedta">EtOH/EDTA sequencing cleanup</a>
	  <a class="dropdown-item" href="#PCR_tips">PCR tips</a>
	  <a class="dropdown-item" href="#PCR_primers">PCR primers</a>
        </div>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#NGS_library_prep">NGS library prep</a>
      </li>
    </ul>
  </div>
</nav>

	    
    <section id="molecular-laboratory-supplies">
<hr>
	  <h3>molecular laboratory supplies</h3>

<p>Here, I'll list some of the main supplies we use in the Lab. Students from our lab or our collaborators' labs can feel free to <a href="https://justinbagley.org/pages/about.html#contact">email me</a> for login information for making online orders from the different vendors in the United States or Brazil. I can also provide email and phone number info that is sometimes necessary for making or checking on purchases from the Brazilian vendors.</p>

	<ul>
		<li><p><a href="http://www.qiagen.com/products/catalog/sample-technologies/dna-sample-technologies/genomic-dna/dneasy-blood-and-tissue-kit">Qiagen DNeasy Blood &amp; Tissue Kit (250)</a>, Qiagen, Catalog No. 69506, typical/last price $665.00.  However, we prefer to buy:  <a href="http://www.qiagen.com/products/catalog/sample-technologies/dna-sample-technologies/genomic-dna/dneasy-blood-and-tissue-kit">Qiagen DNeasy 96 Blood &amp; Tissue Kit (4)</a>, Qiagen, Catalog No. 69581, typical/last price $1,138.00.</p></li>
		<li><p>Invitrogen DNA Extraction Kit.</p></li>
		<li><p><a href="http://www.fishersci.com/ecomm/servlet/fsproductdetail?storeId=10652&amp;productId=1428810&amp;catalogId=29104&amp;matchedCatNo=PRM8298PEX||PRM8298||PRM8291||PRM8296||PRM8295||PRM8297&amp;fromSearch=1&amp;searchKey=gotaq||promega&amp;highlightProductsItemsFlag=Y&amp;endecaSearchQuery=%23store%3DScientific%23nav%3D0%23rpp%3D25%23offSet%3D0%23keyWord%3Dpromega%2Bgotaq%23searchType%3DPROD%23SWKeyList%3D%5B%5D&amp;xrefPartType=From&amp;savings=0.0&amp;xrefEvent=1394736396766_11&amp;searchType=PROD&amp;hasPromo=0">Promega GoTaq Flexi DNA polymerase</a> (500 U), Fisher Scientific, Catalog No. PR-M8295, typical/last price $146.83.</p></li>
		<li><p><a href="http://www.fishersci.com/ecomm/servlet/fsproductdetail?storeId=10652&amp;productId=1437828&amp;catalogId=29104&amp;matchedCatNo=PRM7132||PRM7122||PRM7123||PRM7133&amp;fromSearch=1&amp;searchKey=gotaq||promega&amp;highlightProductsItemsFlag=Y&amp;endecaSearchQuery=%23store%3DScientific%23nav%3D0%23rpp%3D25%23offSet%3D0%23keyWord%3Dpromega%2Bgotaq%23searchType%3DPROD%23SWKeyList%3D%5B%5D&amp;xrefPartType=From&amp;savings=0.0&amp;xrefEvent=1394736396761_9&amp;searchType=PROD&amp;hasPromo=0">Promega GoTaq Master Mix</a> (green), Fisher Scientific, Catalog No. PRM7123 , typical/last price $394.33.</p></li>
		<li><p><a href="http://www.sigmaaldrich.com/catalog/search?interface=All&amp;term=dntp+100&amp;N=0&amp;mode=match%20partialmax&amp;focus=product&amp;lang=en&amp;region=US">Sigma-Aldrich dNTP 100 (100 mM)</a>, Sigma-Aldrich, Catalog No. DNTP100-1KT, typical/last price $282.00.</p></li>
		<li><p>PCR plates (96 well, non-skirted), <a href="http://www.bioexpress.com/divinity-cart/item/452300/GeneMate-Non-skirted-Flat-well-PCR-Plates/1.html">ISC BioExpress Catalog No. T-3184-1 (20 pk)</a> or <a href="http://www.fishersci.com/ecomm/servlet/fsproductdetail?storeId=10652&amp;productId=7165856&amp;catalogId=29104&amp;matchedCatNo=14230236||14230235||14230233||14230234||14230232&amp;fromSearch=1&amp;searchKey=plates||PCR||plate&amp;highlightProductsItemsFlag=Y&amp;endecaSearchQuery=%23store%3DScientific%23nav%3D0%23rpp%3D75%23offSet%3D0%23keyWord%3DPCR%2Bplate%23searchType%3DPROD%23SWKeyList%3D%5B%5D&amp;xrefPartType=From&amp;savings=0.0&amp;xrefEvent=1394737186818_4&amp;searchType=PROD&amp;hasPromo=0">Fisher Scientific Catalog No. 14-230-234 (25 pk)</a>, typical/last price $71.00 to ~$100.00.</p></li>
		<li><p><a href="http://www.bioexpress.com/divinity-cart/item/353945/EXCEL-SCIENTIFIC-Classic-ThermalSeal-Sealing-Films-for-PCR/1.html">ThermaSeal PCR plate sealing films (100 pk)</a>, ISC BioExpress, Catalog No. T-2417-5, typical/last price $71.48 (sale $53.50).</p></li>
		<li>10 ul pipette tips, ISC BioExpress.</li>
		<li>200 ul pipette tips, ISC BioExpress.</li>
		<li>300 ul pipette tips, ISC BioExpress.</li>
		<li>1000 ul pipette tips, ISC BioExpress.</li>
		<li>1.5 mL PCR mix tubes, Fisher Scientific.</li>
		<li><p>0.5 mL PCR/primer tubes, Fisher Scientific.</p></li>
	</ul>
	<p><font size="-1"><a href="#top">top</a></font></p>
	<p><strong> </strong></p>
    </section>


    <section id="DNA-extraction">
<hr>
	  <!-- <p><strong>DNA Extraction Tips &amp; Protocols:</strong></p> -->
	  <h3>dna extraction tips & protocols</h3>

<p>As noted above, I tend to use Qiagen DNeasy Blood &amp; Tissue extraction kits, or Invitrogen kits, to get whole genomic DNA from the fishes and lizards we study, and the starting material is typically white muscle or fin clip tissue.</p>
<p>I prefer to conduct my extractions using 96-well extraction plates with Qiagen DNeasy 96 kits, and the DNeasy 96 protocol for Animal Tissues. I do this wherever possible, because though more expensive this is much easier and faster! However, when working in the Johnson Lab at BYU, we apply some mild modifications to the Qiagen protocol. A copy of the handbook for these Qiagen DNeasy kits is provided free from <a href="http://diagnostics1.com/MANUAL/General_Qiagen.pdf">diagnostics1.com</a> (2006 version).  We modify this Animal Tissues protocol on pgs. 35-40, because the lab centrifuge only reaches up to a max. speed of 4150 x<em>g</em>.  Specifically, to accommodate this, we do our first lysate spin (handbook step #10) for 15 min at 4150 x<em>g</em> (instead of 10 min at 6000 x<em>g</em>), we do the Buffer AW1 spin (handbook step #12) for 7.5-8 min at 4150 x<em>g</em> (instead of 5 min at 6000 x<em>g</em>), and we do the &quot;topless&quot; (no cover) Buffer AW2 spin (handbook step #14) for a first spin for 20 min at 4150 x<em>g</em> (instead of 15 min at 6000 x<em>g</em>) and a second spin of 2 min.  I personally almost always do 1 elution step with 200 ul of water (with 1 min incubation at room temperature), and this has worked very well for me, with very low error rates and reasonable DNA yields.</p>
<p>In Brazil, we are using Invitrogen PureLink® DNA kits to conduct extractions, and these are essentially exactly the same as the Qiagen kits, except the kits employ individual stand-alone columns for each sample, and thus limiting factors include the number of spaces and speed of available centrifuges, the speed with which you (or students) are capable of working on each column, and speed of moving columns between racks and centrifuges. Another difference is that, when using the Invitrogen kits, we always take the extra time to add RNAase to the samples before moving them to the columns.</p>

	<p><font size="-1"><a href="#top">top</a></font></p>
	<p><strong> </strong></p>
    </section>


<a id="PCR_and_Sanger"></a>
<!--    <section id="PCR_and_Sanger">
    </section> 
-->
<hr>
	  <h3>PCR &amp; Sanger sequencing protocols</h3>
<!-- <p><strong>PCR &amp; Sanger Sequencing Protocols </strong></p> -->

<p>What follow are a series of sections containing tips and protocols for conducting PCR and Sanger sequencing. Similar thermocycling routines are useful for other procedures, such as DNA digestion and ligation used during genomic library prepration, e.g. for NGS sequencing experiments.</p>

    <section id="millipore"></section>
<p><strong>PCR Cleanup using Millipore 96 well plates (purple, multiscreen 96)</strong></p>

	<ol style="list-style-type: decimal">
		<li><p>Bring total PCR volume up to 100 μl (for 25 μl reactions that you used 5 μl of for your check gel, you would have ~20–21 μl reaction volume remaining on your PCR plate after completing a check gel analysis; thus, you would need to add ~79–80 μl of autoclaved/dH2O water to bring the reaction volume up to a total of 100 μl).</p></li>
		<li><p>Transfer all 100 μl of diluted PCR reaction to a Millipore 96 well clean up plate.</p></li>
		<li><p>Turn on the Millipore vacuum manifold pump.  Place the Millipore plate on vacuum manifold (with empty or partially empty 96 well collecting plate underneath—check to make sure it is not full) for 10–15 minutes until wells are empty, making sure that the pressure gauge stays at least -24 in Hg (in the Johnson Lab, our Millipore pump usually drops down to -19 Hg; this should be maintained for the entire duration of vacuum filtration).</p></li>
		<li><p>Optional: Cover wells that are not being used if you have problems getting sufficient suction pressure in step 3 above.</p>
	<ol style="list-style-type: decimal">
		<li><p><em>Optional: Cover wells that are not being used if you have problems getting sufficient suction pressure in step 3 above.</em></p></li>
		<li><p><em>Note: the wells of the plate should appear shiny and slightly wet all the time.  The diluted PCR will probably mostly get sucked through the plate fairly quickly, but should be vacuumed for several more minutes after most of the water has disappeared below.</em></p></li>
	</ol></li>
		<li><p>After completing the vacuum filtration step above, turn off the pump and remove the Millipore plate from the vacuum manifold.  Blot the bottom of the plate with a paper towel to remove excess water.</p></li>
		<li><p>Re-suspend clean DNA in 30 μl of autoclaved water added into each well.</p></li>
		<li><p>Place the plate on a vortex / mixer for approximately 8–10 minutes.  Use tape to hold the plate in place and make sure to switch the vortex off and set the dial to 2.5–3 (on a typical vortex machine, e.g. Scientific Industries Vortex Genie 2).</p></li>
		<li><p>Pipette product out of wells and transfer to labeled tube or plate.</p></li>
		<li><p>Alternatively, pipette product out of wells and directly into cycle sequencing reactions.</p></li>
	</ol>

	<p><font size="-1"><a href="#top">top</a></font></p>
	<p><strong> </strong></p>
    </section>


    <section id="exosapit">
<hr>
<p><strong>Faster PCR Cleanup using ExoSAP-IT (Exonuclease I and Shrimp Alkaline Phosphatase in buffer)</strong></p>

	<ol style="list-style-type: decimal">
		<li><p>Place 4 μl of ExoSAP-IT stock into each well containing ~12-15 μl PCR reactions that you desire to clean.</p></li>
		<li><p>Incubate for one cycle at 37 degrees C for 30 minutes to degrade remaining primers and nucleotides.</p></li>
		<li><p>Incubate for one cycle at 80 degrees C for 15 minutes to inactivate ExoSAP-IT.</p></li>
		<li><p>PCR products are now cleaned of main contaminants, and are now ready for DNA sequencing reactions or other down-stream lab work/analyses.</p></li>
	</ol>
	<p><font size="-1"><a href="#top">top</a></font></p>
	<p><strong> </strong></p>
    </section>

    <section id="bigdye">
<hr>
<p><strong>Cycle sequencing reactions using Big Dye v3.0 dye-terminator chemistry</strong></p>

<p>Most labs reduce the amount of Big Dye Terminator Reaction mix they use in each cycle sequencing reaction, in order to make the available mix last longer, and hence save money (Big Dye is expensive!). Dilutions of 1/8 or 1/16 relative to the standard full reaction concentration of 4 μl  of 2.5x Big Dye mix per 10 μl total cycle sequencing reaction volume (or 8 μl per 20 μl total volume) are commonly used. The following table provides cycle sequencing reaction mix setups using each of the common dilutions.</p>
<p><em>1/8th and 1/16th Cycle Sequencing Reaction mix setups</em></p>

<table cellpadding="0" width="443" cellspacing="0" border="1" >
	<tbody >
	<tr >
		<td width="120" valign="top" ><strong>Reaction sizes:</strong></td>
		<td width="111" valign="top" ><strong>1/8th</strong></td>
		<td width="99" valign="top" ><strong>1/16th</strong></td>
		<td width="114" valign="top" ><strong>Modified 1/16th</strong></td>
	</tr>
	<tr >
		<td width="120" valign="top" >H2O</td>
		<td width="111" valign="top" >2.5 μl</td>
		<td width="99" valign="top" >2.75 μl</td>
		<td width="114" valign="top" >2.25 μl</td>
	</tr>
	<tr >
		<td width="120" valign="top" >5x Big Dye buffer</td>
		<td width="111" valign="top" >1.5 μl</td>
		<td width="99" valign="top" >1.75 μl</td>
		<td width="114" valign="top" >1.75 μl</td>
	</tr>
	<tr >
		<td width="120" valign="top" >Big Dye</td>
		<td width="111" valign="top" >1.0 μl</td>
		<td width="99" valign="top" >0.5 μl</td>
		<td width="114" valign="top" >0.5 μl</td>
	</tr>
	<tr >
		<td width="120" valign="top" >Primer (10 μM)</td>
		<td width="111" valign="top" >0.4 μl</td>
		<td width="99" valign="top" >0.4 μl</td>
		<td width="114" valign="top" >0.5 μl</td>
	</tr>
	<tr >
		<td width="120" valign="top" >DNA (10-75 ng/ μl)</td>
		<td width="111" valign="top" >5.0 μl</td>
		<td width="99" valign="top" >5.0 μl</td>
		<td width="114" valign="top" >5.0 μl</td>
	</tr>
	<tr >
		<td width="120" valign="top" ><strong>Total Volume:</strong></td>
		<td width="111" valign="top" ><strong>10.4 μl</strong></td>
		<td width="99" valign="top" ><strong>10.4 μl</strong></td>
		<td width="114" valign="top" ><strong>10 μl</strong></td>
	</tr>
	</tbody>
</table>

	<p>We run our cycle sequencing reactions on ABI 9700 PCR thermocyclers using the following protocol: 95° C for 2 min, followed by 35 cycles of 95° C for 10 s, 52° C for 5 s, and 60° C for 4 min.  Afterward, we use a standard 4° C hold (forever).  In my research, I exclusively use Modified 1/16th mixes for maximum efficiency.</p>
	<p><font size="-1"><a href="#top">top</a></font></p>
	<p><strong> </strong></p>
    </section>

    <section id="sephadex">
<hr>
<p><strong>Sephadex sequencing reaction cleanup</strong></p>

<p>The following DNA cycle sequencing reaction cleanup protocol relies on Sephadex beads from GE Healthcare and is specific to the Johnson Lab at BYU. For example, the protocol contains directions for prepping plates of sequences for submission to the <a href="http://dnasc.byu.edu">BYU DNA Sequencing Center</a> core facility.  Still, the procedure is written in practical terms and can easily be modified to work in almost any molecular evolution or molecular biology lab.</p>

	<ol style="list-style-type: decimal">
		<li><p>Obtain Millipore multiscreen filter plates (96 well, item doesn’t say “PCR”).  Add a used 96-well, v-bottom plate to the bottom of the filter plate.</p></li>
		<li><p>Obtain SephadexTM G-50 fine beads and plate loader.</p></li>
		<li><p>Put plate loader on aluminum foil to catch excess SephadexTM.  Then pour SephadexTM onto plate loader and fill all necessary holes (one for each sample being cleaned).</p></li>
		<li><p>Turn empty filter plate upside down and slide onto (full) plate loader.  Hold firmly, flip the two plates over, and tap plate loader gently (e.g. I use scissors for this) to get the SephadexTM to fall down into each corresponding well.  Place leftover SephadexTM on foil back into container.</p></li>
		<li><p>Add 300 μl dH2O to each well, making sure that the wells are full.  Then get balance plate and fill wells with water, as needed to balance the samples.</p></li>
		<li><p>Let plate stand for 10–15 minutes.  During this time, the water is causing the SephadexTM resin to expand.</p></li>
		<li><p>Spin plates in centrifuge (making sure to use balance plate) for 2 min at 2500 rpm x_g_.</p></li>
		<li><p>Add 20 μl dH2O to sequencing reactions.</p></li>
		<li><p>Dump water from each 96 well plate and place clean, non-balance v-bottomed plates (used in steps 5–7) back into cupboard.</p></li>
		<li><p>Add a new clean 96-well, v-bottom plate underneath each filter plate except the balance plate, making sure rows A–H are aligned on both the filter and collection plates.  Make sure each v-bottom collection plate is labeled with user/lab name and submission number/codes.  Alternatively, if you do not have final submission data, you may simply label the plates according to their order as plates #1, #2, etc. so that you know which plate is which.</p></li>
		<li><p>Transfer all sequencing samples to corresponding filter plates.</p></li>
		<li><p>Spin plates in centrifuge (<em>again, using the same balance plate</em>) for 2 min at 2500 rpm x_g_.</p></li>
		<li><p>Dry samples in vacufuge (vacuum centrifuge) at 60° C for 30–45 min, making sure to use a matching balance plate as needed.  Check your plate(s) at the end of vacufuging, and ensure that all water has been dried down to the bottom of the plate.</p>
	<ol style="list-style-type: decimal">
		<li>Note: you may from time to time forget your plate of samples on the vacufuge… but don’t worry, vacufuging as per above for 1 hour has no adverse effects.</li>
	</ol></li>
		<li><p>Cover plate with thermal seal and mark off all unused wells.  Write DNA sequence submission numbers on each plate, then cover each plate and place in the DNASC (DNA Sequencing Center) fridge for processing (note the correct shelf in the fridge, which will be labeled “Electrophoresis ready”).</p></li>
	</ol>
	<p><font size="-1"><a href="#top">top</a></font></p>
	<p><strong> </strong></p>
    </section>


    <section id="etohedta">
<hr>
<p><strong>Cycle sequencing cleanup using EtOH/EDTA DNA precipitation procedure</strong></p>

<p>For starting CS reaction volumes of 10 μl...</p>

	<ol style="list-style-type: decimal">
		<li><p>Add 2.5 μl of 125 mM EDTA to each sample.</p></li>
		<li><p>Add 27.5 μl of 100% EtOH (cold, from 4 degrees Celsius refrigerator) to each sample.</p></li>
		<li><p>Shake slowly on vortex for a few seconds (say 5-10 seconds for plates; for tubes, invert the tube 4 or 5 times, or buzz once quickly on vortex).</p></li>
		<li><p>Wait 15 minutes.</p></li>
		<li><p>Centrifuge plate at 2,000 xg (rcf) at 4 degrees C for 30 minutes.</p></li>
		<li><p>Remove samples from centrifuge, then cover with a paper towel and invert. Place back in centrifuge and spin for 1 min at 140 x_g_ (rcf).</p></li>
		<li><p>Add 30 μl 70% EtOH (cold, from 4 degrees C refrigerator) to each sample.</p></li>
		<li><p>Centrifuge for 1,650 xg (rcf) at 4 degrees C for 15 minutes.</p></li>
		<li><p>Remove samples from centrifuge, cover with paper towel and invert. Place in centrifuge and spin for 1 min at 140 x_g_ (rcf).</p></li>
		<li><p>Samples are uncovered and mostly dry, but they need to be heated to ensure that they are completely dry. Place samples upright and uncovered into 37-40 degrees C incubator, and dry them there for 15 min. (Alternative: leave tubes/wells upright and uncovered, but optionally with the entire plate loosely covered in aluminum foil, for 1 h.)</p></li>
		<li><p>Resuspend dye-terminator-tagged DNA samples with 10 μl HI-Di Formamide, vortex VERY slowly (or by hand on surface of lab bench) for 10-60 s to mix.</p></li>
		<li><p>Incubate sample at 94 degrees C for 2 min before placing on ABI automatic sequencer.</p></li>
	</ol>
	<p><font size="-1"><a href="#top">top</a></font></p>
	<p><strong> </strong></p>
    </section>

    <section id="PCR_tips">
<hr>
<p><strong>PCR tips  </strong></p>

<p><a href="http://peter.unmack.net/molecular/advice/project.tips.html">Peter Unmack's Molecular Systematics Project Tips</a> - This page containing tips for organizing samples and conducting PCR, clean up, and sequence submissionIn the Johnson Lab, we use a series of PCR data sheets originally developed by Peter Unmack.  Here are links to Peter's example Excel spreadsheets for each step, including files for planning each of your <a href="http://peter.unmack.net/molecular/advice/extraction.july.21.2010.xls">extraction</a>, <a href="http://peter.unmack.net/molecular/advice/pcr.sheet.rain.cytb.jan.21.2012.xls">PCR</a>, <a href="http://peter.unmack.net/molecular/advice/clean.pcr.jan.22.2012.xls">PCR clean up</a>, and <a href="http://peter.unmack.net/molecular/advice/sequencing.sheet.jan.22.2012.xls">sequencing</a> procedure, from <a href="http://peter.unmack.net">peter.unmack.net</a>.  See Peter's website for an example Master list for keeping samples in order. Everyone in the BYU Department of Biology uses our in-house sequencing facility, the BYU DNA Sequencing Center (<a href="http://dnasc.byu.edu">DNASC</a>; run by PhD biochemist Ed Wilcox), for Sanger sequencing on ABI automated sequencers and some NGS sequencing (e.g. Illumina runs).  Thus, for members of the Johnson Lab or Department that have come to this site (or others using DNASC services), I here also provide a sequence submission protocol for submitting plate information to the DNASC.  </p>

	<p><font size="-1"><a href="#top">top</a></font></p>
	<p><strong> </strong></p>
    </section>

    <section id="PCR_primers">
<hr>
<p><strong>PCR primers</strong></p>

<p>What follows is a table of oligonucleotide DNA primers commonly used in my research (all sequences are listed from 5' to 3'), many of which were developed by <a href="http://www.peter.unmack.net">Peter J. Unmack</a> or provided by him to the Johnson Lab, at BYU. In the primer names, capital &quot;L's&quot; or &quot;F's&quot; at the beginning of the name indicate that the primer corresponds to the &quot;light&quot; DNA chain and is a forward primer; capital &quot;H's&quot; or &quot;R's&quot; indicate the primer sits on the &quot;heavy&quot; DNA chain and is a reverse primer. Annealing temperatures of 48 degrees C work well for most of the mitochondrial DNA gene primers listed here, although it is at times advantageous to increase specificity by slightly upping this to 49 degrees C (ad lib.). Typically, the nuclear genes amplify well using annealing temperatures of 52 degrees C, but some are run in nested PCRs. You will want to check the original papers, where referenced (&quot;Source&quot; column), to see what their exact thermoprofiles were. Also see my <a href="http://justinbagley.org/pages/pubs.html">publications</a>, and refs. therein, for more information, including precise amplification conditions that my colleagues and I have used for each primer; particularly good examples of this include Unmack et al. (2012) and Bagley et al. (2013, 2015, 2016).</p>

<table cellpadding="0" width="896" style="font-size: 9.5px;" cellspacing="0" border="0" >
	<tbody >
	<tr >
		<td width="76" height="15" ><strong>Genome</strong></td>
		<td width="77" ><strong>Gene</strong></td>
		<td width="91" ><strong>Primer</strong></td>
		<td width="191" ><strong>Sequence</strong></td>
		<td width="258" ><strong>Source</strong></td>
		<td width="168" ><strong>Taxon</strong></td>
		<td width="35" ><strong>notes</strong></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >L14725</td>
		<td >GAYTTGAARAACCAYCGTTG</td>
		<td >Hrbek et al. (2007)</td>
		<td >Poeciliidae</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >H15982</td>
		<td >CCTAGCTTTGGGAGYTAGG</td>
		<td >Hrbek et al. (2007)</td>
		<td >Poeciliidae</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >HA / OUT(H)</td>
		<td >CAACGATCTCCGGTTTACAAGAC</td>
		<td >Schmidt et al. (1998)</td>
		<td >cyprinids (but work in most taxa)</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >HD / INH</td>
		<td >GGGTTGTTTGATCCTGTTTCGT</td>
		<td >Schmidt et al. (1998)</td>
		<td >cyprinids (but work in most taxa)</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >LA-NEW / OUT(L)</td>
		<td >GTGACTTGAAAAACCACCGTT</td>
		<td >Schmidt et al. (1998), but modified by Peter J. Unmack</td>
		<td >cyprinids (but work in most taxa)</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >LD / INL</td>
		<td >CCATTCGTCATCGCCGGTGC</td>
		<td >Schmidt et al. (1998)</td>
		<td >cyprinids</td>
		<td >
		</td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >H15927</td>
		<td >TTACAAGACCGACGCTCTGA</td>
		<td >Peter J. Unmack</td>
		<td >Poeciliidae</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >Glu31</td>
		<td >TGRCTTGAAAAACCACCGTTGT</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >Glu18</td>
		<td >TAACCAGGACTAATGRCTTGAA</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >Thr29</td>
		<td >ACCTTCGATCTCCTGATTACAAGAC</td>
		<td >Peter J. Unmack</td>
		<td >catfishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >GPThr27</td>
		<td >TCTTCGGATTACAAAACCG</td>
		<td >Peter J. Unmack</td>
		<td >Galaxiella</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >POE.15940</td>
		<td >GACTTTAACCTCCGACTTTCG</td>
		<td >Peter J. Unmack</td>
		<td >Poeciliidae</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >POE.15931</td>
		<td >CCCTCGACTTTCGGTTTACAAG</td>
		<td >Peter J. Unmack</td>
		<td >Poeciliidae</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >Cyt<em>b</em></td>
		<td >RF.Thr.48</td>
		<td >GCAGTAGGAGGGAATTTAACCTTCG</td>
		<td >Peter J. Unmack</td>
		<td >Rainbowfishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" > </td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
	</tr>
	<tr >

		<td height="15" >mitochondrial</td>
		<td >ATPase 6/8</td>
		<td >lys.22F</td>
		<td >AAAGCGTTAGCCTTTTAAGC</td>
		<td >Peter J. Unmack</td>
		<td >Atheriniformes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >ATPase 6/8</td>
		<td >lys.31F</td>
		<td >GCCTTTTAAGCTAAAGATTGG</td>
		<td >Peter J. Unmack</td>
		<td >Atheriniformes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >ATPase 6/8</td>
		<td >co3.62R</td>
		<td >TTATTAGAAGGGCGGCAACTG</td>
		<td >Peter J. Unmack</td>
		<td >Atheriniformes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >mitochondrial</td>
		<td >ATPase 6/8</td>
		<td >co3.23R</td>
		<td >GGCTTGGGTCAACTATGTGGT</td>
		<td >Peter J. Unmack</td>
		<td >Atheriniformes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>RPS7</em></td>
		<td >1F</td>
		<td >TGGCCTCTTCCTTGGCCGTC</td>
		<td >Chow &amp; Hazama (1998)</td>
		<td >most fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>RPS7</em></td>
		<td >2R</td>
		<td >AACTCGTCTGGCTTTTCGCC</td>
		<td >Chow &amp; Hazama (1998)</td>
		<td >most fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>RPS7</em></td>
		<td >3R</td>
		<td >GCCTTCAGGTCAGAGTTCAT</td>
		<td >Chow &amp; Hazama (1998)</td>
		<td >most fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>RPS7</em></td>
		<td >2F</td>
		<td >AGCGCCAAAATACTGAAGCC</td>
		<td >Chow &amp; Hazama (1998)</td>
		<td >most fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>RPS7</em></td>
		<td >2F.2.CAT</td>
		<td >GCCATGTTCAGTACCAGTGC</td>
		<td >Peter J. Unmack</td>
		<td >catfishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>RPS7</em></td>
		<td >3R.2</td>
		<td >CATCTCCAGCTCAAGMAGAG</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>RPS7</em></td>
		<td >3R.10</td>
		<td >TCAGAGTTCATCTCCAGCTC</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>RPS7</em></td>
		<td >1F.2</td>
		<td >CTCTTCCTTGGCCGTCGTTG</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>RPS7</em></td>
		<td >2R.67</td>
		<td >TACCTGGGARATTCCAGACTC</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>RPS7</em></td>
		<td >3R.24</td>
		<td >AGCTGAGCCTTCAGGTCAGAG</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>src</em> (TK)</td>
		<td >src.e7.1F</td>
		<td >TGACAGACGTTTGTCCCGTACTGAAGC</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>src</em> (TK)</td>
		<td >src.e7.2F</td>
		<td >GTACTGAAGCCTCAGACTCAGG</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>src</em> (TK)</td>
		<td >src.e8.1F</td>
		<td >CTGAAGCCTGGCACCATGTC</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>src</em> (TK)</td>
		<td >src.e8.2F</td>
		<td >CACCATGTCCCCTGAGGCKTTCCTGC</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>src</em> (TK)</td>
		<td >src.e8.R</td>
		<td >ACGGCATAAAGCTGAACCAG</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>src</em> (TK)</td>
		<td >src.e10.endR</td>
		<td >ATGAGKCGAGCCAGACCGAAATCAGC</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>src</em> (TK)</td>
		<td >src.e10.end2R</td>
		<td >CCGAAATCAGCCACTTTACAMACCAG</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>yes</em> (TK)</td>
		<td >yesA</td>
		<td >GATCGCAGACGGCATGGCCTCC</td>
		<td >Schartl et al. (1995)</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>yes</em> (TK)</td>
		<td >yesB</td>
		<td >CTGCCCTTTGTTACCAGCTCTGTC</td>
		<td >Schartl et al. (1995)</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>yes</em> (TK)</td>
		<td >yes.F1</td>
		<td >GAGAGAATGAACTACATCCATAG</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>yes</em> (TK)</td>
		<td >yes.F2</td>
		<td >GACAACCTGGTCTGTAAGATCGC</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>yes</em> (TK)</td>
		<td >yes.R1</td>
		<td >GACCACACGTCTGATTTGATTGTGAA</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td >X-<em>yes</em> (TK)</td>
		<td >yes.R2</td>
		<td >GATTTGATTGTGAAGCGACCGTACA</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>Rag1</em></td>
		<td >RAG1.f409.mod</td>
		<td >AGGTGYTTGTGCCGTCTCTGTGG</td>
		<td >Peter J. Unmack</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>Rag1</em></td>
		<td >Rag1.F74</td>
		<td >TTTCGGAATGGAAGTTTAAGCTSTTTCG</td>
		<td >Sullivan et al. (2006)</td>
		<td >catfishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>Rag1</em></td>
		<td >Rag1.F89</td>
		<td >TTTAAGCTGTTTCGAGTTCGTTCATTGG</td>
		<td >Peter J. Unmack</td>
		<td >catfishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" >nuclear</td>
		<td ><em>Rag1</em></td>
		<td >R1.4090R</td>
		<td >CTGAGTCCTTGTGAGCTTCCATRAAYTT</td>
		<td >Guillermo Orti</td>
		<td >fishes</td>
		<td ></td>
	</tr>
	<tr >
		<td height="15" ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
		<td ></td>
	</tr>
</tbody>
</table>


<p><em>Primer References (&quot;Source&quot; column in primer table above)</em></p>

<p>Chow S, Hazama K (1998) Universal PCR primers for S7 ribosomal protein gene introns in fish. Molecular Ecology 7:1255–1256.</p>
<p>Hrbek T, Seekinger J, Meyer A (2007) A phylogenetic and biogeographic perspective on the evolution of poeciliid fishes. Mol. Phylogenet. Evol. 43:986-998.</p>
<p>Schartl M, Wilde B, Schlupp I, Parzefall J (1995) Evolutionary origin of a parthenoform, the Amazon molly <em>Poecilia formosa</em>, on the basis of molecular genealogy. Evolution 49:827-835.</p>
<p>Schmidt TR, Bielawski JP, Gold JR (1998) Molecular phylogenetics and evolution of the cytochrome b gene in the cyprinid genus Lythrurus (Actinopterygii: Cypriniformes). Copeia 1998:14-22.</p>
<p>Sullivan JP, Lundberg JG, Hardman M (2006) A phylogenetic analysis of the major groups of catWshes (Teleostei: Siluriformes) using rag1 and rag2 nuclear gene sequences. Molecular Phylogenetics and Evolution 41:636-662.</p>

	<p>[More coming soon, including primers for <em>ldh-A</em> (muscle-type lactose dehydrogenase; Quattro and Jones 1999) and <em>Glyt</em> (glycosyltransferase; Li et al. 2007) that Justin used during his dissertation work.]</p>
	<p><font size="-1"><a href="#top">top</a></font></p>
	<p><strong> </strong></p>
    </section>


    <section id="NGS_library_prep">
<hr>
<h3>NGS library prep</h3>
<p><strong>Protocols for double digest RAD-seq (ddRad-seq) library preparation</strong></p>

<p>If you don't want to bother with library prep, see my blog entry <a href="http://justinbagley.rbind.io/2013/11/27/outsource-next-generation-sequencing-runs-de-novo-snp-discovery-phylogenetics-phylogeography/">here</a> for links to companies/labs for outsourcing single and dd RAD-seq runs for <em>de novo</em> SNP discovery.</p>
<p>During my postdoctoral training, I have learned to prepare genomic libraries for generating Rad-tags (Rad SNP loci) using double-digest RAD-seq (ddRAD-seq) experiments with Next-Generation Sequencing (NGS) platforms. With our Brazilian collaborators, we have been using an altered version of the Peterson et al. (2012) ddRAD protocol, modified for the Ion Torrent platform, because this is faster and cheaper for us that using other platforms or outsourcing. We use similar methods to prep libraries and conduct ddRad-seq runs on the Illumina platform in the <a href="https://www.eckertlab.com">Eckert Lab</a>. The main difference between prepping ddRAD-seq libraries for Ion Torrent runs versus Illumina Mi-Seq or Hi-Seq runs is that these two different platforms use different sets of primers! So, no big deal really; but you can get as fancy with the primers and adapters as you would like. </p>
<p>Here, I am posting links to RAD-family methods, but focusing on ddRAD protocols, and I plan to expand this list in the future as I come across new protocols. </p>
<p>For in-house RAD-seq library prep and sequencing, perhaps the most currently used protocol is the &quot;original&quot; ddRad-seq protocol of <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0037135">Peterson et al. (2012)</a>. A variety of different evolutionary genetics/population genomics labs have taken up this protocol and modified it for their particular study systems/organisms or NGS machines. I have found it very helpful to look over the protocols that these labs have posted freely online. Below, I give links to a few such examples from various institutions, as follows:</p>

<ul>
	<li><p><a href="../pdf/DNA_library_prep_(parchman).pdf">Andrew Eckert Lab ddRAD-seq protocol</a> (modified from Parchman et al. 2012; see three-step procedure at bottom of <a href="https://github.com/EckertLab/protocols">README</a> at eckertlab protocols GitHub repository; <a href="https://github.com/EckertLab/protocols/blob/master/DNA_library_prep_(parchman).md">Markdown</a>)</p></li>
	<li><p><a href="https://datadryad.org/bitstream/handle/10255/dryad.37448/rfseq_protocol.pdf?sequence=1">Parchman et al. (2012) ddRAD-seq protocol</a> (from <a href="https://datadryad.org//resource/doi:10.5061/dryad.m2271pf1">Dryad accession</a> for the corresponding publication)</p></li>
	<li><p><a href="https://github.com/DrK-Lo/lab-wiki/wiki/5---ddRAD-library-prep-(Alan)">Katie Lotterhos Lab ddRAD-seq bench protocol</a> (from <a href="https://github.com/DrK-Lo/lab-wiki/wiki">GitHub lab-wiki Home</a>)</p></li>
	<li><p><a href="http://openwetware.org/wiki/Sacks:RAD-seq">Erik Sacks UIUC Dept. of Crop Sciences lab ddRAD-seq protocol</a></p></li>
	<li><p><a href="http://mcdaniellab.biology.ufl.edu/files/ddRad_seq_protocol_McDaniel_lab-150611.pdf">ddRAD-seq protocol from McDaniel Lab at University of Florida</a></p></li>
	<li><p><a href="http://edenrcn.com/protocols/Individual%20Protocols/Wasik_butterfly_RAD_tagging.pdf">Butterfly ddRAD-seq protocol</a> </p></li>
</ul>

	<p><font size="-1"><a href="#top">top</a></font></p>
	<p><strong> </strong></p>
    </section>


<p><strong>References</strong></p>

<p>Parchman TL, Gompert Z, Mudge J, Schilkey FD, Benkman CW, Buerkle CA (2012) Genome‐wide association genetics of an adaptive trait in lodgepole pine. Molecular Ecology, 21(12), 2991-3005. <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1365-294X.2012.05513.x">HTML</a></p>
<p>Peterson BK, Weber JN, Kay EH, Fisher HS, Hoekstra HE (2012) Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species. PLoS ONE, 7, e37135. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0037135">HTML</a></p>

<p><font size="-1"><a href="#top">top</a></font></p>
<hr>

        
    </div>
    <!-- /.container -->

    <!-- Footer -->
    <footer class="py-5 bg-dark">
      <div class="container">
        <p class="m-0 text-center text-white">Copyright &copy; 2018 Justin C. Bagley</p>
      </div>
      <!-- /.container-narrow -->
    </footer>

    <!-- Bootstrap core JavaScript -->
    <script src="../vendor/jquery/jquery.min.js"></script>
    <script src="../vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

  </body>

</html>
