<!DOCTYPE html>
<html lang="en">

  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>research</title>

    <!-- Bootstrap core CSS -->
    <link href="../vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="../css/1-col-portfolio.css" rel="stylesheet">

  </head>

  <body>

    <!-- Navigation -->
    <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
      <div class="container">
        <a class="navbar-brand" href="../index.html">Justin C. Bagley - Ecological & Evolutionary Genetics</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
          <ul class="navbar-nav ml-auto">
            <li class="nav-item">
              <a class="nav-link" href="../index.html">home</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="about.html">about</a>
            </li>
            <li class="nav-item active">
              <a class="nav-link" href="research.html">research
                <span class="sr-only">(current)</span>
              </a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="pubs.html">publications</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="talks.html">talks</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="software.html">software</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="teaching.html">teaching</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" href="protocols.html">protocols</a>
            </li>
          </ul>
        </div>
      </div>
    </nav>


<!-- Page Content -->
    <div class="container">
      <div></div>
      <p></p>

<!-- Research Content - Overview -->
    <div class="container-narrow">
      <h1 class="my-4">research</h1>

<nav class="navbar navbar-expand-lg navbar-light bg-light">
  <a class="navbar-brand" href="#">Navbar</a>
  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button>
  <div class="collapse navbar-collapse" id="navbarNavDropdown">
    <ul class="navbar-nav">
<!--      <li class="nav-item active">
        <a class="nav-link" href="#research-interests-overview">Overview<span class="sr-only">(current)</span></a>
      </li>
-->
      <li class="nav-item">
        <a class="nav-link" href="#systematics-phylogenomics-and-species-delimitation">Phylogenomics</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#molecular-phylogeography-freshwater-terrestrial-ecosystems">Phylogeography</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#genomics-of-speciation-adaptation-and-climate">Speciation & Adaptation</a>
      </li>
<!--
      <li class="nav-item">
        <a class="nav-link" href="#code-bioinformatics-and-software-pipelines">Code & Bioinformatics</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#research-projects-summary">Projects</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#phd-dissertation-research-info">Dissertation</a>
      </li>
-->
      <!-- OPTIONAL: DROPDOWN MENU BUTTON -->
      <li class="nav-item dropdown">
	<a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Other</a>
        <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
	  <a class="dropdown-item" href="#code-bioinformatics-and-software-pipelines">Code & Bioinformatics</a>
	  <a class="dropdown-item" href="#research-projects-summary">Projects</a>
	  <a class="dropdown-item" href="#phd-dissertation-research-info">Dissertation</a>
        </div>
      </li>
    </ul>
  </div>
</nav>

<section id="research-interests-overview">
  <hr>
<p>I am an evolutionary biologist focused on problems in molecular ecology and biodiversity science. My research program primarily focuses on the use of genome-wide SNP and targeted sequence capture data to infer the evolutionary relationships of species and populations, and to understand how historical/ecological processes have shaped the distributions of biodiversity in space and time. My focus on biodiversity leads inevitably to the main target areas for biodiversity worldwide–the Neotropical biodiversity ‘hotspots’ (e.g. defined <a href="https://www.nature.com/articles/35002501">here</a>), which harbor the majority of Earth’s plant and animal species. My current research topics are diverse, reflecting my diverse interests and skillset, and include projects focused on: </p>
<p><ul>–<strong>1. Phylogenomics of rapid angiosperm radiations</strong> with Andean-centered distributions, and that of their bat pollinators (Tropical Andes hotspot)</ul>
<ul>–<strong>2. <a href="http://www.oxfordbibliographies.com/view/document/obo-9780199941728/obo-9780199941728-0041.xml">Phylogeography</a></strong> of temperate and Neotropical plants (e.g. forest trees) and freshwater fishes (Brazilian Cerrado and Mesoamerica hotspots), and</ul>
<ul>–<strong>3. Understanding the interplay of <a href="https://www.nature.com/scitable/knowledge/library/speciation-the-origin-of-new-species-26230527">speciation</a>, <a href="https://en.wikipedia.org/wiki/Adaptation">adaptation</a>, and gene flow (hybridization)</strong> in outcrossing forest trees that form hybrid zones (North America, South America, China).</ul>
</p>
<p>Each area of my research program has benefitted from rapid advances in high-throughput sequencing (HTS) methods and bioinformatics tools over the last few years. Bioinformatically, this work is computationally intensive (multiple programming languages and simulation frameworks, plus genetic and ecological niche modeling approaches) and requires handling genome-scale data from species with genomes of widely variable size and complexity (∼1 to >10 gigabases), as well as dealing with polyploidy issues in plants.</p>
<p><strong><em>Click on the gray navigation bar above to quickly access research topics below.</em></strong></p>
<br>
<!--<p>I am interested in how historical and ecological processes act to shape the spatial and temporal distributions of biodiversity in freshwater and terrestrial environments. Much of my work has focused on <strong>1)</strong> using <strong>molecular <a href="http://www.oxfordbibliographies.com/view/document/obo-9780199941728/obo-9780199941728-0041.xml">phylogeography</a> and species delimitation</strong> to better understand population structure and history, species limits, and the role of historical biogeographical processes in shaping species distributions and community assembly in Neotropical biodiversity ‘<a href="https://en.wikipedia.org/wiki/Biodiversity_hotspot">hotspots</a>’. However, high-throughput sequencing methods and bioinformatics have advanced rapidly, providing tools that I have actively used during the last few years to improve my phylogeographic and phylogenetic inferences and expand my research into <strong>2)</strong> disentangling the interplay between genetic drift, natural selection, and gene flow (hybridization) during <strong><a href="https://www.nature.com/scitable/knowledge/library/speciation-the-origin-of-new-species-26230527">speciation</a> and local <a href="https://en.wikipedia.org/wiki/Adaptation">adaptation</a></strong>. This latter focus includes studying the ecological and genetic bases of adaptation to challenging environments and ecological gradients, and will contribute to future genome-wide association study (GWAS) approaches combining genomic and phenotypic datasets to understand the genetic architecture of ecologically important traits.</p>
<p>Current projects focus on <strong>evolutionary and ecological genomics</strong> of North American and Neotropical <em>freshwater fishes</em> (tetras, catfishes, cichlids, and livebearers) and <em>forest trees</em> (white pines, quaking aspen, and Chilean pine). Within the Neotropics, I am mainly interested in uncovering drivers of biodiversity in the Mesoamerica (Central America) and Brazilian Cerrado biodiversity 'hotspots' (e.g. defined <a href="https://www.nature.com/articles/35002501">here</a>).</p>
<p><strong><em>Click on the lower navigation bar above to quickly access different topics below.</em></strong></p>
-->
</section>

<section id="systematics-phylogenomics-and-species-delimitation">
  <hr>
<!--<h3>systematics, phylogenomics, &amp; species delimitation</h3> -->
<h3>phylogenomics &amp; species delimitation</h3>
<!-- <p><img src="../img/Burmeistera_PI_fig.png" title="Burmeistera seq capture phylogenetic informativeness" alt="Burmeistera phylogenetic informativeness" width="300"/>  <img src="../img/catostomidae_pi_profiling_over_mcc_tree.png" title="Catostomidae phylogenetic informativeness" alt="Catostomidae phylogenetic informativeness" width="300"/></p> -->
<p><img src="../img/Burmeistera_tanglegram.png" title="Burmeistera phylogenomics tanglegram" alt="Burmeistera phylogenomics tanglegram" width="430"/>  <img src="../img/Burmeistera_PI_fig.png" title="Burmeistera seq capture phylogenetic informativeness" alt="Burmeistera phylogenetic informativeness" width="210"/></p>
<p>My research in systematics and taxonomy primarily focuses on discovering and determining evolutionary relationships among species, and clarifying species limits, through analyses of DNA sequence data and morphological characters. Hypotheses of phylogenetic relationships are reconstructed based on broad taxon and character sampling using maximum-likelihood and Bayesian inference analyses, relaxed molecular clocks, clock-partitioning schemes, and tip- and fossil-calibrated models (e.g. fossilized birth-death process). Our work also seeks to describe new biodiversity to improve biodiversity accounting. This work focuses on understanding diversification and the evolution of adaptive traits (e.g. reproductive mode) in monophyletic clades of temperate and Neotropical plants (bellflowers, pines), freshwater fishes, and squamate reptiles. Recent <a href="https://justinbagley.org/pages/pubs.html">papers</a> focus on </p>
	<p><ul>–Phylogenomics of recent, rapid angiosperm radiations with Andean-centered distributions, using <em>Burmeistera</em> (Campanulaceae) as a case study</ul>
	<ul>–Integrating molecules, morphology, and fossils to infer relationships among North American <a href="https://en.wikipedia.org/wiki/Catostomidae">'sucker' fishes (Catostomidae)</a></ul>
	<ul>–Integrative taxonomy and species delimitation in Cerrado lizards.</ul></p>
<p><font size="-1"><a href="#top">top</a></font></p>
</section>

<section id="molecular-phylogeography-freshwater-terrestrial-ecosystems">
  <hr>
<h3>phylogeography of freshwater &amp; terrestrial ecosystems</h3>
<table class="wide">
<tr>
  <td class="left">  
<img src="../img/aspen_popstruct_fig1.png" alt="Aspen GBS phylogeography" title="Aspen GBS phylogeography" width="500" class="center"/>
</td>
</tr>
</table>
<p><strong>Phylogeography</strong>, the study of the geographical distributions of genetic lineages within and among closely related species, has assumed a central role in molecular ecology for inferring the historical and ecological processes influencing present-day biodiversity patterns. Along with collaborators from the U.S., Brazil, and Mexico, I have been involved in a number of projects using single-species and comparative (multispecies) phylogeography approaches to improve our understanding of the diversification, stability, and assembly of North American and Neotropical biotic communities. This work currently focuses on three intriguing systems that present different contexts for understanding evolutionary processes: <strong>1)</strong> North American forest trees, and freshwater fish assemblages of the <strong>2)</strong> Mesoamerica (Central America) and <strong>3)</strong> Brazilian Cerrado biodiversity 'hotspots'.</p>
<p>Here is a more in-depth look at my <em>prior</em> research in <a href="phylogeog.html">phylogeography</a>.</p>
<p><font size="-1"><a href="#top">top</a></font></p>
</section>

<section id="genomics-of-speciation-adaptation-and-climate">
  <hr>
<h3>genomics of speciation &amp; local adaptation, with implications for climate change</h3>
<table class="wide">
<tr>
  <td class="left">
<img src="../img/swwp_figb_pop_struct.png" alt="ddRADseq population structure example (Menon et al. 2018)" title="ddRADseq population structure example (Menon et al. 2018)" width="350"/>
</td>
  <td class="right">
<img src="../img/swwp_fig4_genomic_cline.png" alt="genomic cline example (Menon et al. 2018)" title="genomic cline example (Menon et al. 2018)" width="400"/>
</td>
</tr>
</table>
<!-- swwp_fig5_dem.png -->

<p>Understanding how barriers to gene flow arise, and new species boundaries are formed and maintained, is a central goal of evolutionary biology. Yet, the study of '<a href="https://global.oup.com/ushe/product/speciation-9780878930890?cc=us&amp;lang=en&amp;">speciation</a>', presents numerous challenges and requires integrative perspectives taking into account geographical factors (biogeography) and range dynamics, niche evolution, reproductive isolation, as well as the evolution of neutral and adaptive genetic variation in the context of demographic history. I have become interested in the ways in which plant versus animal systems can reveal different aspects of the speciation process.</p>
<p>Our current projects on <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.14505">southwestern white pine</a> (<em>Pinus strobiformis</em>) highlight the role of hybridization and extrinsic factors during ecological speciation. We are also using genome scan and environmental association analyses to understand local adaptation to climatic gradients and challenging environments (high elevation) in this system, with important implications for understanding tree responses to <em>climate change</em>. By contrast, some Neotropical fish lineages that I have studied (e.g. livebearers; Poeciliidae) are more predisposed to speciation through intrinsic pre- or postmating isolation, and exhibit presumed niche conservatism but a high degree of phenotypic variation. I am eager to develop future projects expanding my speciation and adaptation work along these two avenues of research.</p>
<p><font size="-1"><a href="#top">top</a></font></p>
</section>

<section id="code-bioinformatics-and-software-pipelines">
  <hr>
<h3>code &amp; bioinformatics pipelines</h3>
<!-- The rapid growth of high-throughput sequencing technologies and their use in molecular ecology 
has precipitated a need for practical code and software solutions to the challenges presented by 
the resulting deluge of genome-wide DNA sequence data. -->
<p>Check my <a href="https://github.com/justincbagley">GitHub</a> and <a href="https://justinbagley.org/pages/software.html">software</a> pages to see what I've been coding lately. However, the main software package that I am currently developing is <a href="https://github.com/justincbagley/piranha">PIrANHA v0.3a2</a>, a useful repository of pipelines and wrapper functions for automating tasks in phylogenetics and phylogeography.</p>
<!-- <p>[More soon.]</p> -->
<p><font size="-1"><a href="#top">top</a></font></p>
</section>

<section id="research-projects-summary">
  <hr>
<p>What follows is a slightly dated summary of projects I am involved in, as recently as 2017–2018, which I hope to find time to update soon.</p>
<iframe src="//www.slideshare.net/slideshow/embed_code/key/ComxCAB9CevHkU" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> 
<strong> <a href="//www.slideshare.net/justinbagley/current-projects-summary" title="Current Projects Summary" target="_blank">Current Projects Summary</a> </strong> from <strong><a href="https://www.slideshare.net/justinbagley" target="_blank">Justin C. Bagley</a></strong>
</div>
<p><font size="-1"><a href="#top">top</a></font></p>
</section>

<section id="phd-dissertation-research-info">
  <hr>
<h3 id="phd-dissertation-research">PhD dissertation research</h3>
<p>[coming soon.]</p>
<p><font size="-1"><a href="#top">top</a></font></p>
</section>
      <hr>

      </div>
    <!-- /.container-narrow -->


    </div>
    <!-- /.container (Page Content) -->

    <!-- Footer -->
    <footer class="py-5 bg-dark">
      <div class="container">

<p align="left"><script>
  (function() {
    var cx = '001889442856820716750:0-utetvkkfu';
    var gcse = document.createElement('script');
    gcse.type = 'text/javascript';
    gcse.async = true;
    gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
    var s = document.getElementsByTagName('script')[0];
    s.parentNode.insertBefore(gcse, s);
  })();
</script>
<gcse:search></gcse:search>
</p>

<!--
        <p class="m-0 text-center text-white">Copyright &copy; 2018 Justin C. Bagley</p> 
-->
        <p class="m-0 text-left text-white"><a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a></p>
        <p class="m-0 text-left text-white"><font size="-2">This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.</font></p>
        <p class="m-0 text-left text-white"><font size="-2">See <a rel="policies" href="https://justinbagley.org/policies.html">policies</a> for terms and conditions.</font></p>
      </div>
      <!-- /.container -->
    </footer>

    <!-- Bootstrap core JavaScript -->
    <script src="../vendor/jquery/jquery.min.js"></script>
    <script src="../vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

  </body>

</html>

