
var tipuesearch = {"pages": [
	{"title":"Justin C. Bagley - Ecological & Evolutionary Genetics", "text": "Justin C. Bagley - Ecological & Evolutionary Genetics Justin C. Bagley - Ecological & Evolutionary Genetics home (current) about research publications talks software teaching tutorials cv github blog @justincbagley Search More figures Copyright © 2018 Justin C. Bagley", "img": "", "url": "https://justinbagley.org/index.html"},
	{"title":"about", "text": "about Justin C. Bagley - Ecological & Evolutionary Genetics home about (current) research publications talks software teaching tutorials cv github blog @justincbagley Search Justin C. Bagley, Ph.D. Evolutionary Biologist, Postdoc Virginia Commonwealth University University Research Interests Phylogeography Population genomics Systematics \/ Diversification Bioinformatics Education Ph.D., Integrative Biology, Brigham Young University M.Sc., Biology, The University of Alabama B.Sc., Biology, The University of Alabama about Justin C. Bagley is a Postdoctoral Scholar in the Department of Biology at Virginia Commonwealth University; Senior Research Associate in the Departamento de Zoologia at Universidade de Brasília; researcher in ecological and evolutionary genetics of animals and plants in North America and the Neotropics; developer of PIrANHA, RADish, and other repositories for analyzing population genomic and phylogenomic datasets. After receiving his BS and MSc degrees in Biology from The University of Alabama in 2004 and 2008, he received his PhD in Integrative Biology in 2014, from Brigham Young University for his work on comparative phylogeography and species delimitation in Central American freshwater fishes under Jerry Johnson. He was a Young Talent Fellow postdoc in Brazil's CNPq Science Without Borders program under Francisco Langeani and Guarino Colli at the Universidade de Brasília, 2015–2017. In 2017, he moved to Virginia Commonwealth University, where he is now a Postdoctoral Scholar in Andrew Eckert's Plant Evolutionary Genetics Laboratory. Justin is an evolutionary biologist focusing on problems in molecular ecology and biodiversity science – particularly the use of genome-wide SNP data to understand how historical and ecological processes shape the spatial and temporal distributions of biodiversity in freshwater and terrestrial environments. Current research projects focus on molecular phylogeography, speciation, and local adaptation in North American and Neotropical freshwater fishes and forest tree communities. Justin is a reviewer for Molecular Ecology, Molecular Phylogenetics and Evolution, PeerJ, Proceedings of the Royal Society B, and a dozen other journals in ecology and evolutionary biology. curriculum vitae orcid: 0000-0001-6737-8380 google scholar contact Justin Bagley Plant Evolutionary Genetics Laboratory Department of Biology Virginia Commonwealth University 1000 W Cary St Richmond, Virginia 23284-2012 USA Email: jcbaIgleydon't@wantvcuspam! .So please leave me alone!edu Phone: +1 (804)-828-0820 Copyright © 2018 Justin C. Bagley", "img": "", "url": "https://justinbagley.org/pages/about.html"},
	{"title":"research", "text": "research Justin C. Bagley - Ecological & Evolutionary Genetics home about research (current) publications talks software teaching tutorials research I am interested in how historical and ecological processes act to shape the spatial and temporal distributions of biodiversity in freshwater and terrestrial environments. Much of my work has focused on 1) using molecular phylogeography and species delimitation to better understand population structure and history, species limits, and the role of historical biogeographical processes in shaping species distributions and community assembly in Neotropical biodiversity ‘hotspots’. However, high-throughput sequencing methods and bioinformatics have advanced rapidly, providing tools that I have actively used during the last few years to improve my phylogeographic and phylogenetic inferences and expand my research into 2) disentangling the interplay between genetic drift, natural selection, and gene flow (hybridization) during speciation and local adaptation. This latter focus includes studying the ecological and genetic bases of adaptation to challenging environments and ecological gradients, and will contribute to future genome-wide association study (GWAS) approaches combining genomic and phenotypic datasets to understand the genetic architecture of ecologically important traits. Current projects focus on evolutionary and ecological genomics of North American and Neotropical freshwater fishes (tetras, catfishes, cichlids, and livebearers) and forest trees (white pines, quaking aspen, and Chilean pine). Within the Neotropics, I am mainly interested in uncovering drivers of biodiversity in the Mesoamerica (Central America) and Brazilian Cerrado biodiversity 'hotspots' (e.g. defined here). Click on the lower navigation bar above to quickly access different topics below. phylogeography of freshwater & terrestrial ecosystems Phylogeography, the study of the geographical distributions of genetic lineages within and among closely related species, has assumed a central role in molecular ecology for inferring the historical and ecological processes influencing present-day biodiversity patterns. Along with collaborators from the U.S., Brazil, and Mexico, I have been involved in a number of projects using single-species and comparative (multispecies) phylogeography approaches to improve our understanding of the diversification, stability, and assembly of North American and Neotropical biotic communities. This work currently focuses on three intriguing systems that present different contexts for understanding evolutionary processes: 1) North American forest trees, and freshwater fish assemblages of the 2) Mesoamerica (Central America) and 3) Brazilian Cerrado biodiversity 'hotspots'. Here is a more in-depth look at my prior research in phylogeography. top genomics of speciation & local adaptation, with implications for climate change Understanding how barriers to gene flow arise, and new species boundaries are formed and maintained, is a central goal of evolutionary biology. Yet, the study of 'speciation', presents numerous challenges and requires integrative perspectives taking into account geographical factors (biogeography) and range dynamics, niche evolution, reproductive isolation, as well as the evolution of neutral and adaptive genetic variation in the context of demographic history. I have become interested in the ways in which plant versus animal systems can reveal different aspects of the speciation process. Our current projects on southwestern white pine (Pinus strobiformis) highlight the role of hybridization and extrinsic factors during ecological speciation. We are also using genome scan and environmental association analyses to understand local adaptation to climatic gradients and challenging environments (high elevation) in this system, with important implications for understanding tree responses to climate change. By contrast, some Neotropical fish lineages that I have studied (e.g. livebearers; Poeciliidae) are more predisposed to speciation through intrinsic pre- or postmating isolation, and exhibit presumed niche conservatism but a high degree of phenotypic variation. I am eager to develop future projects expanding my speciation and adaptation work along these two avenues of research. top systematics, phylogenomics, & species delimitation My research in systematics and taxonomy primarily focuses on discovering and determining evolutionary relationships among species, and clarifying species limits, through analyses of DNA sequence data and morphological characters. Hypotheses of phylogenetic relationships are reconstructed based on broad taxon and character sampling using Bayesian inference analysis, relaxed molecular clocks, clock-partitioning schemes, and tip- and fossil-calibrated models (e.g. fossilized birth-death process). Our work also seeks to describe new biodiversity to improve biodiversity accounting. This work focuses on understanding diversification and the evolution of adaptive traits (e.g. reproductive mode) in monophyletic clades of freshwater fishes, squamate reptiles, and pine trees. Recent papers focus on integrating molecules, morphology, and fossils to infer relationships among North American 'sucker' fishes (Catostomidae); and integrative taxonomy and species delimitation in Cerrado lizards. top code & bioinformatics pipelines [coming soon.] top Current Projects Summary from Justin C. Bagley top PhD dissertation research [coming soon.] top Copyright © 2018 Justin C. Bagley", "img": "", "url": "https://justinbagley.org/pages/research.html"},
	{"title":"phylogeography", "text": "phylogeography Justin C. Bagley - Ecological & Evolutionary Genetics home about research (current) publications talks software teaching tutorials phylogeography what is phylogeography? [G]eology and biogeography are both parts of natural history and, if they represent the independent and dependent variables respectively in a cause and effect relationship, … they can be reciprocally illuminating. –Rosen (1978, p. 776) [C]omparative phylogeographic assessments within each of multiple codistributed species … offer perhaps the greatest hope for significant advances in understanding how … the demographic and natural histories of populations, can influence intraspecific phylogeographic patterns. –Avise (1998, pp. 376-377) In the past, molecular biology informed the methods of population genetics, while biogeographers operated independently of these fields. However, since the 20th Century molecular genetics revolution in systematics and population genetics, it is the tools of these fields that dictate methods for analyzing genetic data from natural populations, and this in turn has sparked a new field of ‘molecular biogeography.’ The growth of the ‘new’ biogeography is widely acknowledged to have particularly benefited from the rapid development of phylogeography. Yet, what is phylogeography? Phylogeography is a relatively young and integrative field of science that uses molecular data to infer the processes influencing geographical distributions of genetic lineages within and among species, especially at the intraspecific level (Avise et al. 1987; Avise 2000). The phylogeography literature base has grown remarkably fast over the nearly three decades since the birth of the field, yielding many insights into the histories of biotas in different ecosystems across the globe, including cryptic and often temporally ‘deep’ genealogical divergences within many species ranges (e.g. reviewed by Beheregaray 2008; Knowles 2009). By linking these patterns of population divergence with data on geographical barriers, earth history events, distribution models, and speciation processes, phylogeography permits identifying and testing whether and over which temporal scales historical and recurrent processes have shaped intraspecific diversification in an area. Indeed, the reciprocal illumination that Rosen (1978; quoted above) envisioned for vicariance biogeography can also be achieved through phylogeography, as phylogeography provides means of inferring how species responded to geological processes without reliance on phylogenetic structure or areas of endemism required by other historical biogeography methods (Zink 2002); for example, historical processes from the unobservable past can still be inferred even when genetic breaks revealed by selectively neutral markers do not correspond to any present-day geographical barrier. Moreover, because population or species demographic histories influence the shapes of their gene genealogies, we can use coalescent theory (a stochastic, backwards-in-time theory of genealogical processes; Kingman 1982) to estimate their underlying genealogies (e.g. by simulating a distribution of them) as well as population parameters describing their histories (e.g. past changes in population sizes) while accounting for various population genetic processes (e.g. subdivision, speciation, mutation, recombination; reviewed by Kuhner 2009). An important paradigm shift in phylogeography has been the realization that phylogeographic inferences are improved by developing and discriminating among competing coalescent-based demographic models using statistical methods that account for the stochasticity of genetic processes (Knowles & Maddison 2002; Knowles 2009; Nielsen & Beaumont 2009). The above graphic (above-center) summarizes several 'paths' to phylogeographic inference reviewed by Bloomquist et al. (2010), including comparative, spatial diffusion, and statistical population genetics approaches, which each view the recovery of species demographic histories from different perspectives involving different forms of inferences (e.g. pattern-based methods; diffusion-model based inferences; and full-likelihood, full-Bayesian, and ABC model-based analyses). my prior research in Phylogeography Given I have discussed my ongoing research in phylogeography elsewhere in my About, Research, and Collaborators pages, this section provides a summary of my prior work in this fascinating and growing field, including studies conducted along two main lines of research: congeneric and comparative phylogeography model-based inference of species responses to Pleistocene climate and sea level change congeneric and comparative phylogeography While working towards my Master’s in Biology at UA, I asked what unique insights congeneric phylogeographical sampling (within a genus) might yield into understanding the phylogenetics and species limits of North American black basses (Centrarchidae: Micropterus), an iconic group of game fishes with a complex systematic and taxonomic history_. Based on sampling from across the geographical ranges of black bass species, we found multiple cases of mitochondrial introgression\/hybridization events and a novel Gulf Coast clade representing a new lineage to science; thus phylogeography reveals a more complex view of black bass evolution highlighting population-level processes (Bagley et al. 2011, Biol. J. Linn. Soc.). I also combined mtDNA phylogeography, microsatellite DNA population genetics, and morphometric analysis to study Alabama Bass (M. henshalli). I found that, despite evidence for hybridization with Redeye Bass (M. coosae), M. henshalli is overall genetically and morphologically distinct, particularly within the Tallapoosa River system (manuscript in prep.). Comparative phylogeography provides a basis for more robust historical inferences (e.g. of historical biogeographical scenarios for a region) by searching for replicated geographical patterns across multiple codistributed species (e.g. Bermingham and Martin 1998), yielding insights into community assembly and diversification not possible through single-species phylogeography alone. My Ph.D. dissertation work at BYU focused on using an integrative approach coupling comparative phylogeography with species delimitation to improve our understanding of the biogeography and species limits in complex tropical landscapes. The Central American (CA) land bridge and its freshwater stream fish assemblage provide an ideal study system for this research combining a complex\/recent physical history with rich geomorphology and high endemism, including several codistributed lineages of freshwater fishes. My dissertation (1) developed and tested biogeographical hypotheses for CA through a synthesis of geological and phylogeographical data (Bagley & Johnson 2014a, Biol. Rev.); (2) analyzed single-species\/lineage phylogeography datasets for up to eight lineages, including Poeciliidae (five lineages), Characidae (two species), and Cichlidae (one lineage, 4 spp.); and (3) ultimately scaled my own data and data curated from online repositories up to comparative analyses of 8 fish species\/lineages (Bagley & Johnson 2014b, Ecol. Evol.; other manuscripts in prep.). Given a moderate to high degree of taxonomic uncertainty for various groups of Neotropical fishes I studied, this work also evolved to entail a component of species delimitation. Specifically, I used mollies in the Poecilia sphenops species complex_ _(Poeciliidae) as a model system for delimiting species and studying processes of diversification in widespread, morphologically conserved groups of Neotropical fishes, through coalescent-based analyses of multilocus genetic datasets. Therefore, the over-arching goals of my Ph.D. were to evaluate the impacts of historical processes (e.g. geological, climatic events) on geographical distributions of genetic lineages, to understand the relationship between species ecological differences and their phylogeographic histories, and to test and refine Central American freshwater fish species boundaries. I also worked with my Ph.D. advisor, Jerry Johnson, to build phenotypic datasets that will allow me to compare morphometric and life history variation with phylogeographic divergence using spatial-pattern analyses, to test hypotheses about selection and species limits. While at BYU, I also developed research on the biogeography and evolution of Australian freshwater fishes in collaboration with Peter Unmack. By melding species tree analyses with phylogeographic inferences from mitochondrial and nuclear loci, we documented an Oligocene-Miocene origin for most species of galaxiid fishes in the genus Galaxiella (Galaxiidae), high genetic diversity, and strong evidence for a cryptic species within the threatened dwarf galaxias, G. pusilla (Unmack et al. 2012, PLoS ONE). Our findings weigh importantly on conserving this threatened clade of fishes, e.g. long-term planning should preserve multiple populations with unique alleles, in addition to the multiple ancient species in the group. I am currently working to expand my work in Australia to help develop a comparative perspective on the phylogeographic diversification of Australian freshwater fish assemblages. With Dr. Unmack, I am also presently working to expand phylogenetic systematics work on iconic game fishes, through a multilocus phylogenetic analysis of temperate perches in the family Percichthyidae, which are the largest fish species historically present in most temperate Australian freshwater habitats. model-based inference of species responses to Pleistocene climate and sea level change A product of the past two decades of analytical revolutions in biology has been the development of novel model-based statistical tools for (1) inferring parameters of ecological niches and predicting species past, present and future distributions, and (2) using coalescent simulations to arrive at objective rather than ad hoc inferences of processes influencing species phylogeographic histories. I am interested in using these toolsets, i.e. ecological niche modeling and statistical phylogeography, to test hypotheses and infer population history (e.g. lineage spread and range shifting), particularly the responses of species from unglaciated subtropical and tropical areas to climatic fluctuations of the late Pleistocene glaciations. For example, in our work on Galaxiella fishes, we documented a shared demographic-growth response across taxa associated with relaxation of aridity during the late Pleistocene, based on statistical comparisons of Bayesian demographic models using Bayes factors (Unmack et al. 2012, PLoS ONE). I then revisited the southeastern U.S. freshwater fish fauna to test the potential impact of expansion-contraction cycles on species ranges, hence patterns of genetic structure, during the last glacial cycle. Paleoclimatic modeling, phylogeography, and coalescent simulations using sequence data from least killifish, Heterandria formosa, suggested that this cold-intolerant species contracted its range to a peninsular Florida refugium during the Last Glacial Maximum (Bagley et al. 2013, BMC Evol. Biol.). We also found evidence for nuclear-mitochondrial discordances indicating evidence for historical gene flow and sex-biased migration. future phylogeography research directions While the review of my previous work on phylogeography above is informative, it is exciting to speculate about potential future research directions in this field. One thing I am very excited about regarding the future of my phylogeographical research is that I intend to continue the past lines of research above, while maintaining a ‘big picture’ view, by focusing on community and ecosystem-level syntheses. In addition, I look forward to developing the phylogeography arm of my research program along several lines of research, including: using model-based comparative phylogeography to infer community assembly and diversification, using landscape genetics and ecological niche modeling to explore the relative roles of ecology and geography in shaping genetic isolation in natural populations (with empirical analyses of Central American terrestrial and freshwater species), and using conservation phylogeography to aid resolving the 'Linnean' and 'Wallacean' shortfalls in biogeography and conservation, and to inform conservation management and policy, particularly in the Central American and South American Neotropics. While I intend to conduct this research in my main geographical areas of focus, the South American (Brazil) and Central American Neotropics, I would be interested in pursuing conservation biogeography in my other study systems discussed on my Research page, or any other region of the world with a highly endemic and threatened biota. However, by extending phylogeography to inform management I will help us not only understand, but also conserve, the patterns and processes of diversification in unique Neotropical (and other) regions of the world. top References Avise JC (1998) The history and purview of phylogeography: a personal reflection. Molecular Ecology, 7, 371-379. Avise JC (2000) Phylogeography: the History and Formation of Species. Harvard University Press, Cambridge, Massachusetts. Avise JC, Arnold J, Ball RM, Bermingham E, Lamb T, Neigel JE, Reeb CA, Saunders NC (1987) Intraspecific phylogeography: the mitochondrial DNA bridge between population genetics and systematics. Annual Review of Ecology and Systematics, 18, 489-522. Bagley JC, Johnson JB (2014a) Phylogeography of the lower Central American Neotropics: diversification between two continents and between two seas. Biological Reviews, 89(4), 767-790. Bagley JC, Johnson JB (2014b) Testing for shared biogeographic history in the lower Central American freshwater fish assemblage using comparative phylogeography: concerted, independent, or multiple evolutionary responses? Ecology and Evolution, 4(9), 1686-1705. Bagley JC, Mayden RL, Roe KJ, Holznagel W, Harris PM (2011). Congeneric phylogeographical sampling reveals polyphyly and novel biodiversity within black basses (Centrarchidae: Micropterus). Biological Journal of the Linnean Society, 104(2), 346-363. Bagley JC, Sandel M, Travis J, Lozano-Vilano M de L, Johnson JB (2013) Paleoclimatic modeling and phylogeography of least killifish, Heterandria formosa: insights into Pleistocene expansion-contraction dynamics and evolutionary history of North American Coastal Plain freshwater biota. BMC Evolutionary Biology, 13, 223. Unmack PJ, Bagley JC, Adams MD, Hammer MD, Johnson JB (2012) Molecular phylogeny and phylogeography of the Australian freshwater fish genus Galaxiella (Teleostei: Galaxiidae), with an emphasis on dwarf galaxias (G. pusilla). PLoS One, 7(6), e38433. Beheregaray LB (2008) Twenty years of phylogeography: the state of the field and the challenges for the Southern Hemisphere. Molecular Ecology, 17, 3754-3774. Bermingham E, Martin AP (1998) Comparative mtDNA phylogeography of neotropical freshwater fishes: testing shared history to infer the evolutionary landscape of lower Central America. Molecular Ecology, 7, 499-517. Bloomquist EW, Lemey P, Suchard MA (2010) Three roads diverged? Routes to phylogeographic inference. Trends in Ecology & Evolution, 25, 626-632. Knowles LL (2009) Statistical phylogeography. Annual Review in Ecology, Evolution, and Systematics, 40, 593-612. Rosen DE (1978) Vicariant patterns and historical explanation in biogeography. Systematic Zoology, 27, 159-188. Zink RM (2002) Methods in comparative phylogeography, and their application to studying evolution in the North American aridlands. Integrative and Comparative Biology, 42, 953-959. top Copyright © 2018 Justin C. Bagley", "img": "", "url": "https://justinbagley.org/pages/phylogeog.html"},
	{"title":"publications", "text": "publications Justin C. Bagley - Ecological & Evolutionary Genetics home about research publications (current) talks software teaching tutorials pubs To download the publications below, click on the Adobe PDF logo following each reference and obtain PDF files from the publisher's website or my ResearchGate profile. However, to comply with copyright laws, I ask that you please limit the use of PDF files to printing of single copies for personal research and study, and do not modify the files in any way or use them for commercial purposes. Thanks. ~ J Follow this link to see my Google Scholar Citations. journal articles 2018 Bagley JC, Aquino PPU, Hrbek T, Hernandez S, Langeani F, Colli GR (in revision) Using ddRAD-seq phylogeography to test for genetic effects of headwater river capture in suckermouth armored catfish (Loricariidae: Hypostomus) from the central Brazilian Shield. Molecular Ecology. Bagley JC, Mayden RL, Harris PM (2018) Phylogeny and divergence times of suckers (Cypriniformes: Catostomidae) inferred from Bayesian total-evidence analyses of molecules, morphology, and fossils. PeerJ, 6, e5168. HTML Menon MG, Bagley JC, Friedline C, Whipple A, Schoettle A, Sáenz AL, Wehenkel CA, Flores-Renteria LH, Sneizko R, Cushman S, Waring K, Eckert AJ (2018) The role of hybridization during ecological divergence between southwestern white pine, Pinus strobiformis, and limber pine, P. flexilis. Molecular Ecology, 27, 1245-1260. Breitman MF, Domingos FMCB, Bagley JC, Wiederhecker HC, Ferrari TB, Cavalcante VHGL, et al. (accepted) New species of Enyalius (Squamata: Leiosauridae) endemic to the Brazilian Cerrado, with comments on how to improve the practice of taxonomy. Herpetologica. 2017 Overcast I, Bagley JC, Hickerson, MJ (2017) Strategies for improving approximate Bayesian computation tests for synchronous diversification. BMC Evolutionary Biology, 17, 2013. HTML 2016 Bagley JC, Matamoros WA, McMahan CD, Tobler M, Chakrabarty P, Johnson JB (2016) Phylogeography and species delimitation in convict cichlids (Cichlidae: Amatitlania): implications for taxonomy and Plio–Pleistocene evolutionary history in Central America. Biological Journal of the Linnean Society. 2015 Bagley JC, Alda F, Breitman MF, van den Berghe EP, Bermingham E, Johnson JB (2015) Assessing species boundaries in a morphologically conserved group of Neotropical freshwater fishes, the Poecilia sphenops species complex (Poeciliidae). PLoS One, 10(4), e0121139. HTML 2014 Watson CM, Makowsky R, Bagley JC (2014) Reproductive mode evolution in lizards revisited: updated analyses examining geographic, climatic and phylogenetic effects support the cold-climate hypothesis. Journal of Evolutionary Biology, 27(12), 2767-2780. HTML Bagley JC, Johnson JB (2014) Testing for shared biogeographic history in the lower Central American freshwater fish assemblage using comparative phylogeography: concerted, independent, or multiple evolutionary responses? Ecology and Evolution, 4(9), 1686-1705. HTML Bagley JC, Johnson JB (2014) Phylogeography of the lower Central American Neotropics: diversification between two continents and between two seas. Biological Reviews, 89(4), 767-790. HTML 2013 Bagley JC, Sandel M, Travis J, Lozano-Vilano M de L, Johnson JB (2013) Paleoclimatic modeling and phylogeography of least killifish, Heterandria formosa: insights into Pleistocene expansion-contraction dynamics and evolutionary history of North American Coastal Plain freshwater biota. BMC Evolutionary Biology, 13, 223. doi: HTML 2012 Unmack PJ, Bagley JC, Adams MD, Hammer MD, Johnson JB (2012) Molecular phylogeny and phylogeography of the Australian freshwater fish genus Galaxiella (Teleostei: Galaxiidae), with an emphasis on dwarf galaxias (G. pusilla). PLoS One, 7(6), e38433. HTML 2011 Bagley JC, Mayden RL, Roe KJ, Holznagel W, Harris PM (2011). Congeneric phylogeographical sampling reveals polyphyly and novel biodiversity within black basses (Centrarchidae: Micropterus). Biological Journal of the Linnean Society, 104(2), 346-363. HTML proceedings and book chapters Johnson JB, Bagley JC (2011) Ecological drivers of life-history evolution. In: Ecology and evolution of poeciliid fishes. (eds Evans JP, Pilastro A, Schlupp I), pp. 38-49. University of Chicago Press, Chicago, IL. ISBN-10: 0226222748. (NOTE: This chapter was peer-reviewed.) Book Thompson M, Bagley JC, Rasco J, Lackey K, Cox M (2007) The Respiratory System. In: Biology: The Study of Life (eds Rasco J, Lackey K) Kendall\/Hunt Publishing. ISBN-10: 0757556434. dissertation Bagley JC (2014) Understanding the diversification of Central American freshwater fishes using comparative phylogeography and species delimitation. PhD dissertation, Department of Biology, Brigham Young University, Provo, UT. Abstract short communications Ceríaco LM, Gutiérrez EE, Dubois A, et al. (Appendix 1 of Supporting Signatories including JC Bagley) (2016) Photography-based taxonomy is inadequate, unnecessary, and potentially harmful for biological sciences. Zootaxa, 4196, 435-445. HTML doi: 10.11646\/zootaxa.4196.3.9. editorials and letters [coming soon.] Copyright © 2018 Justin C. Bagley", "img": "", "url": "https://justinbagley.org/pages/pubs.html"},
	{"title":"talks", "text": "talks Justin C. Bagley - Ecological & Evolutionary Genetics home about research publications talks (current) software teaching tutorials talks 2018 Menon MG, Sáenz ALG, Landguth E, Bagley JC, Schoettle A, Wehenkel CA, Cushman S, Waring K, Eckert AJ (2018) Tracing the footprints of a hybrid zone under a history of divergence with gene flow. 21st Annual Graduate Research Symposium and Exhibit, Virginia Commonwealth University, Richmond, VA, April 24. (University Symposium; Poster Presentation) Bagley JC, Menon MG, Friedline C, Whipple A, Schoettle A, Sáenz ALG, Wehenkel CA, Flores-Rentería LH, Sniezko R, Cushman S, Waring K, Eckert AJ (2018) Population genomics and paleoclimatic modeling support speciation with gene flow, not genomic islands of differentiation, in sky-island populations of southwestern white pine. International Plant & Animal Genome Conference XXVI, San Diego, CA, January 13–17. (International Meeting; Poster Presentation) 2017 Bagley JC, Aquino PPU, Hrbek T, Hernandez S, Langeani F, Colli GR (2017) Using ddRAD- seq phylogeography to test for genetic effects of headwater river capture in suckermouth armored catfish (Loricariidae: Hypostomus) from the central Brazilian Shield. Evolution Meeting 2017, Portland, OR, June 23–27. (International Meeting; Oral Presentation) Bagley JC, Menon MG, Friedline C, Whipple A, Schoettle A, Sáenz ALG, Wehenkel CA, McGarvey D, Flores-Rentería LH, Sniezko R, Cushman S, Waring K, Eckert AJ (2017) Population genomics and paleoclimatic modeling support speciation with gene flow, not genomic islands of differentiation, in sky-island populations of southwestern white pine. Evolution Meeting 2017, Portland, OR, June 23–27. (International Meeting; Poster Presentation) Menon MG, Bagley JC, Friedline C, Whipple A, Schoettle A, Sáenz ALG, Wehenkel CA, McGarvey D, Flores-Rentería LH, Sniezko R, Cushman S, Waring K, Eckert AJ (2017) What’s in a name? Ecological speciation in southwestern white pine. Evolution Meeting 2017, Portland, OR, June 23–27. (International Meeting; Oral Presentation) Ferrari TBU, Breitman MF, Domingos F, Wiederhecker HC, Bagley JC, de Mello R, de Lima TG, Colli G (2017) Taxonomia integrative e delimitação de espécies no Cerrado: uma nova espécie de Enyalius (Squamata: Leiosauridae) endemic do bioma. VIII Congresso Brasileiro de Herpetologia, Mato Grosso, Brazil. (National Meeting; Abstract + Oral Presentation) Wiederhecker HC, Breitman MF, Domingos F, Bagley J, Colli G (2017) Next-Generation Teaching: ensinando ao pesquisar taxonomia integrative beneficia estudantes e professores. VIII Congresso Brasileiro de Herpetologia. (National Meeting; Abstract + Oral Presentation) 2015 Aquino PPU, Bagley JC, Couto TBA, Figueira-Soares YFU (2015) Sinal biogeográfico de captura de cabeceira: primeiro registro de Phalloceros harpagos Lucinda, 2008 (Cyprinodontiformes: Poeciliidae) em riachos da bacia do rio São Francisco. III Simpósiode Zoologia Sistemática, UFMG Campus Pampulha, Belo Horizonte, Minas Gerais, Brazil, December 16–20. (Regional Meeting; Abstract + Poster Presentation) 2014 Unmack PJ, Bagley JC, Davis A, Hammer MP, Adams M, Johnson JB (2014) Phylogeny, biogeography and evolution of the temperate perches (Percichthyidae). Australian Society for Fish Biology meeting, Darwin Convention Center, Darwin, Northern Territory, Australia, June 30–July 4. (National Meeting; Abstract + Poster Presentation) Bagley JC, Alda FA, Breitman MF, van den Berghe EP, Bermingham E, Johnson JB (2014) Who’s your “molly”? Species boundaries and cryptic diversity in a complex of Neotropical freshwater fishes. BYU Grad Expo, Brigham Young University, Provo, UT, U.S.A., March 25. (Invited Poster Presentation) Bagley JC (2014) Paleoclimatic modeling and phylogeography of least killifish, Heterandria formosa: insights into Pleistocene expansion-contraction dynamics along the North American Coastal Plain. Center for Bioenvironmental Research, Tulane University, New Orleans, Louisiana, U.S.A., January 13. (Invited Oral Presentation) 2013 Bagley JC, Johnson JB (2013) Phylogeography of the lower Central AmericanNeotropics: diversification between two continents, between two seas. Joint International Meeting of Ichthyologists and Herpetologists, Albuquerque, New Mexico, U.S.A., July 9–15. (Interna- tional Meeting; Abstract + Oral Presentation) Bagley JC, Sandel M, Travis J, Lozano-Vilano M de L, Johnson JB (2013) Testing biogeographic hypotheses of Plio—Pleistocene history and diversification along the North American Gulf-Atlantic Coastal Plain in least killifish (Heterandria formosa). Evolution Conference, Snowbird Resort, Salt Lake City, Utah, U.S.A., June 20–25. (International Meeting; Poster presentation) 2012 Bagley JC (2012) Integrative comparative phylogeographical approaches to understanding evolutionary diversification in Central American freshwater fishes. São Paulo School for Advanced Science-Evolution (SPSAS-evo), Ilhabela, Brazil, August 19–31. (International Work- shop; Poster Presentation) Bagley JC, Johnson JB (2012) Comparative phylogeography of Central American freshwater fishes. 1st Joint Congress on Evolutionary Biology, Ottawa, Ontario, Canada, July 6–10. (International Meeting; Poster Presentation) 2010 Lozano-Vilano M de L, Bagley JC (2010) A new species of Heterandria from Coahuila state, México. Annual Meeting of the Texas Academy of Sciences, Austin, Texas, U.S.A. (Abstract) Bagley JC, Johnson JB (2010) Do secondary freshwater fishes show congruent responses to saltwater barriers? Evolution 2010, Joint Annual Meeting of the Society for the Study of Evolution, Society of Systematic Biologists, and American Society of Naturalists, Portland, Oregon, U.S.A., June 25–29. (International Meeting; Oral Presentation) 2009 Nay LU, Bagley JC, Johnson JB (2009) Life history variation in the knife-edged livebearer, Alfaro cultratus. Joint International Meeting of Ichthyologists and Herpetologists, Port- land, Oregon, U.S.A., July 22–27. (International Meeting; Abstract + Poster Presentation) Bagley JC (2009) Phylogenetic comparative analyses of body size and shape variation in a diverse yet morphologically conservative clade of lungless salamanders (Plethodontidae: Desmognathus). Department of Biology Ecolunch Seminar, Brigham Young University, Provo, Utah, U.S.A., January 30. (Oral Presentation) 2008 Bagley JC, Brummer T, Nay L, Johnson JB (2008) Phylogeography, geometric morphometrics, and life history variation of Alfaro cultratus (Cyprinodontiformes: Poeciliidae) from Costa Rica. Desert Fishes Council Meeting, Cuatro Ciénegas, Mexico, November. (Annual Meeting; Abstract + Poster Presentation) Bagley JC (2008) Polyphyly, hybridization and cryptic biodiversity among Micropterus. Desert Fishes Council Meeting, Cuatro Ciénegas, Mexico, November. (Annual Meeting; Abstract + Oral Presentation) Bagley JC (2008) A new molecular phylogeny of the black basses, Micropterus (Teleostei: Centrarchidae), based on increased within-taxon sampling. Department of Biology Ecolunch Seminar, Brigham Young University, Provo, Utah, U.S.A., October 31. (Oral Presentation) Bagley JC, Harris PM (2008) Taxonomy, population genetics and body shape variation in Alabama spotted bass. Joint International Meeting of Ichthyologists and Herpetologists, Montreal, Quebec, Canada. (International Meeting; Abstract + Oral Presentation) 2004 Bagley JC, Secor SM (2004) Allometry of digestive performance in the marine toad and diamondback water snake. Society for Integrative and Comparative Biology Meeting, San Diego, California, U.S.A. (Annual Meeting; Abstract + Poster Presentation) Copyright © 2018 Justin C. Bagley", "img": "", "url": "https://justinbagley.org/pages/talks.html"},
	{"title":"software", "text": "software Justin C. Bagley - Ecological & Evolutionary Genetics home about research publications talks software (current) teaching tutorials software PIrANHA Scripts for file processing and analysis in phylogenetics and phylogeography. NOTE This software repository, despite several minor-version releases, is still under development with the first major-version release expected sometime later this year. MAGNET Shell script pipeline for inferring ML gene trees for many loci (e.g. genomic RAD loci). GaussClust Clustering using Gaussian mixture modeling for species delimitation and classification. MissingDataFX Testing effects of missing data on phylogenetic inferences. RAPFX Sensitivity analysis of RADseq assembly parameter effects on downstream genetic analyses. GBSequenceMiner Tool for mining GenBank accession numbers & DNA sequences from manuscript PDFs Copyright © 2018 Justin C. Bagley", "img": "", "url": "https://justinbagley.org/pages/software.html"},
	{"title":"teaching", "text": "teaching Justin C. Bagley - Ecological & Evolutionary Genetics home about research publications talks software teaching (current) tutorials teaching Below you will find a brief but informative overview of my teaching experience. Far greater detail can be found in my CV. I also provide, on a separate page, a variety of links to Teaching Resources that I have encountered, and that many readers and educators will find of interest. Teaching Summary from Justin C. Bagley NOTE: Links may only work when viewing from SlideShare. If slides are blurry, then it may help to wait a few seconds for them to load, and then open them in full screen mode. Copyright © 2018 Justin C. Bagley", "img": "", "url": "https://justinbagley.org/pages/teaching.html"},
	{"title":"teaching resources", "text": "teaching resources Justin C. Bagley - Ecological & Evolutionary Genetics home about research publications talks software teaching (current) tutorials teaching resources Resources for Developing Lecture Courses Everyone wants to \"design smart lectures.\" Here are links to some nice short videos from the University of Minnesota Center for Teaching and Learning (CTL) Mini Workshop on designing lectures that can help us do this: video set 1 - Examples, video set 2 - Overview (to be watched in this order). Requires Adobe Flash 6 plug-in. While designing lectures, it is also important to understand the questions and objectives of education. One way to develop your understanding of these topics is to read about [Bloom's Taxonomy](http:\/\/en.wikipedia.org\/wiki\/Bloom's_taxonomy) and be aware of ways to facilitate the highest levels of learning--analysis, synthesis, evaluation, and creation--in the classroom. 10 steps for building your first academic course Developing a Course - Teaching Commons, DePaul University Design & Teach a Course, a guide within the Carnegie Mellon University Eberly Center, with two main parts: Develop Your Course Teach Your Course Active Learning Techniques Since becoming a buzzword in education in the 1980s and 1990s, \"active learning\" remains a hot topic at all levels of educational literature and practice, being implemented in primary through university teaching. Active learning essentially refers to any and all teaching methods that engage students to think about the material without passively listening to lecture (or multimedia) content. Several research studies suggest that active learning is better than simply lecturing students, and many educators including the majority of university professors use active learning techniques as a more effective way of \"guiding\" students towards \"learning goals\". Consistent with the immense popularity of active learning, there are numerous resources online that can help university professors design and implement active learning methods. Here is a short list of links to resources that I have found and used thus far in my career. Active Learning Techniques at the Brigham Young University Center for Teaching and Learning (CTL) Active learning on Wikipedia Tips for building interactivity during lectures from the Carnegie Mellon University Eberly Center The Field-tested Learning Assessment Guide (FLAG) is also a great site to visit. This website provides detailed information on a variety of active learning techniques, including concept mapping, minute papers, and portfolios. Useful resources for specific course types in the biological sciences Ecology course content and syllabi online Biology 1B 2010, Ecology section, video playlist (Lectures 1 - 13), University of California-Berkeley Phil Ganter BIOL 4120 Principles of Ecology course, Tennessee State University Mark Pyron BIO 216 Ecology course, Ball State University, Indiana Stephen C. Stearns Principles of Evolution, Ecology and Behavior course syllabus Open Yale courses Yale Univ. Mark Leighton Fundamentals of Ecology course syllabus at Harvard Extension Schoolcourse: http:\/\/isites.harvard.edu\/icb\/icb.do?keyword=k98171syllabus: http:\/\/isites.harvard.edu\/icb\/icb.do?keyword=k98171&pageid=icb.page615110 Thomas M. Gehring General Ecology course, Central Michigan University Yale Geology course with climate and background on life on earth\/paleontology useful to ecology students - Links to syllabus and lectures as pdfs University of Michigan Global Change class with good info that could be useful, particularly those on human ecology Copyright © 2018 Justin C. Bagley", "img": "", "url": "https://justinbagley.org/pages/teachingres.html"}
]};
